
\documentclass{amsart}
\usepackage{amsmath,amsfonts,amssymb,amsthm}
\usepackage[english]{babel}
\usepackage{graphicx}
\usepackage{url}
\usepackage{palatino}
\usepackage[round]{natbib}
\newcommand{\forarxiv}[1]{#1}
\newcommand{\notforarxiv}[1]{}

% show keys for eqs, etc.
% \usepackage[notref,notcite]{showkeys}

% code
\newcommand{\program}{\textsf{program}}

\newcommand{\eat}[1]{}

\newcommand{\FIGmassTransport}{\
\begin{figure}[ht]
\begin{center}
  \forarxiv{\includegraphics[width=13cm]{mass_transport.pdf}}
\end{center}
\caption{\
  Caption goes here.
}
\label{FIGmassTransport}
\end{figure}
}
\newcommand{\refFIGmassTransport}{1}

\hyphenation{Ge-nome Ge-nomes Me-ta-ge-nome Me-ta-ge-nomes Ma-cro-ev-o-lu-tion-ary}

% Noah suggests making a table for
%alpha diversity
%beta diversity
%PCA
%taxonomic assignment/classification
%clustering/binning

\begin{document}

\notforarxiv{
\begin{flushright}
Version dated: \today
\end{flushright}
\bigskip
\noindent RH: PHYLOGENETICS AND THE HUMAN MICROBIOME
\bigskip
\medskip
\begin{center}

\noindent{\Large \bf Phylogenetics and the human microbiome.}
\bigskip

\noindent {\normalsize \sc
Frederick A. Matsen IV$^1$}\\
\noindent {\small \it
$^1$
Program in Computational Biology, Fred Hutchinson Cancer Research Center, Seattle, WA, 91802, USA}\\
\end{center}
\medskip
\noindent{\bf Corresponding author:} Frederick A Matsen, Program in Computational Biology, Fred Hutchinson Cancer Research Center, Seattle, WA, 91802, USA; E-mail: matsen@fhcrc.org.\\
\vspace{1in}
}

\forarxiv{\
\title{Phylogenetics and the human microbiome.}
\author{Frederick A. Matsen IV}
\date{\today}
\begin{abstract}
}
\notforarxiv{
\subsubsection{Abstract}
}

The human microbiome is the collection of microbes that live inside and on the outside of humans.
Recent work including the Human Microbiome Project (HMP) and the Metagenomics of the Human Intestinal Tract (MetaHIT) consortia has advanced human microboime research by generating thousands of 16s surveys and terabases of metagenomic data on the human microbiome.
In this paper I review the impact of phylogenetics and tree-thinking on the methods used in the analysis of human microbiome data, as well as describing current challenges for phylogenetics coming from this type of work.

\forarxiv{
\end{abstract}
\maketitle
}

\notforarxiv{
\noindent (Keywords: human microbiome; microbial ecology; phylogenetic methods; review)\\
\vspace{1.5in}
}


\section{Introduction}

The parameter regime and focus of microbiome research sits outside of the traditional setting of eukaryotic phylogenetics, thus why should our community be interested in what these microbial ecologists and medical researchers have done?
This system is data- and question-rich.
It absolutely requires molecular methods, as microbes can be identified by their molecular sequence, which is much more straightforward to do in high throughput than morphological or phenotypic characterization.
Indeed, just to do ecology you have to do sequencing.

Although there appears to be an artificial divide between phylogeny as practiced as part of microbial ecology and that for multicellular organisms, however there are many parallels between the two enterprises.
Both communities struggle with issues of sequence alignment, large-scale tree reconstruction, species definition.
However, approaches differ between the microbial ecology community and that of eukaryotic phylogenetics, in part because the former has access to a fantastic diversity of organisms directly through culture-independent methods, leading to additional problems above usual.
The species concept is problematic for microbes and there is also considerable discussion concerning how to group them into species-like units.
The diversity of these organisms is great and organizing them into a taxonomy is a serious challenge, especially in the absence of obvious morphological features.

Because of challenges with species definitions, microbial ecology researchers have developed their own techniques for doing comparing samples which are explicitly phylogenetic rather than requiring species concepts.
Although there is some overlap with previous literature, these techniques are novel and could be used in a wider setting and deserve wider consideration by the phylogenetics community.

The human microbiome part of microbial ecology is specifically interesting because questions of microbial genomics, translated into questions of function, have important consequences for human health.
Additionally, due to more than a century of hospital lab medicine, our knowledge about human-associated microbes is much richer than about microbes in general.
It is relatively routine to manipulate the human microbiome or models thereof via intervention studies and germ-free animals.

In this review I intend to inform the \textit{Systematic Biology} readership of phylogenetics-related research happening in microbial ecology and contrast approaches between the two community.
I will use \textit{eukaryotic phylogenetics} to indicate what I think of as the mainstream of SB readership, and \textit{microbial phylogenetics} to denote the other.
I realize that there is substantial overlap-- for instance the microbial community is very interested in unicellular fungi-- but we will use this terminology with the associated caveats.
There is of course also substantial overlap in methodology, however as we will see there are significant differences in approach and the two areas have developed somewhat in parallel.
I will first briefly review the recent literature on the human microbiome, then describe novel ways in which human microbiome researchers have used trees and the challenges that human microbiome research poses for the phylogenetics community.
I will finish with opportunities for the \textit{Systematic Biology} audience to contribute to this field.
I firmly believe that both fields would benefit from more interactions.
I have made explicit effort to make a neutral comparison between two areas; indeed, microbial phylogenetics requires algorithms and ideas that work in parameter regimes orders of magnitude larger than typical for eukaryotic phylogenetics.

\section{The human microbiome}
The human microbiome is the collection of microbial organisms that live inside of and on the surface of humans.
These organisms are populous: it has been estimated that there are ten times as many bacteria associated with each individual than there are human cells.
The microbiome has remarkable metabolic potential, with a set of genes estimated to be about 150 times larger than the human collection of genes \citep{qin2010human}.
Much of our metabolic interaction with the outside world is mediated by our microbiome, as it has important roles in both vitamin
% http://www.sciencedirect.com/science/article/pii/S095816691200119X
and drug \citep{maurice2013xenobiotics} metabolism; our food and drug intake also impacts the diversity of microbes present.
In this section I will briefly review what is known about the human microbiome and its effect on our health.

The human microbiome is an ecosystem.
The microbiome is dynamic in terms of representation but apparently constant in terms of function \citep{hmp2012structure}.
The ``core'' microbiome, or the microbiome shared between all humans, has been described \citep{turnbaugh2008core}.
Human microbiome is spatially organized, as can be seen on skin \citep{grice2009topographical}, with substantial variation in human body habitats across space and time \citep{costello2009bacterial}.
There is a substantial range of inter-individual versus intra-individual variation \citep{hmp2012structure}.

Our actions can shift the composition of our microbiome.
Changes in diet very quickly shift its composition, and there is a strong correlation between long-term diet and microbiome \citep{li2009human,wu2011linking}.
Antibiotics fundamentally disturb microbial communities, resulting in an effect that lasts for years \citep{jernberg2007long,dethlefsen2008pervasive,jakobsson2010short,dethlefsen2011incomplete}.

Considerable attention has been given to how the microbiome influences host phenotype.
IBD, etc, NNN.
kwashiorkor \cite{smith2013gut}.
There is strong evidence of interaction between the gut microbiome and obesity, but the story is yet clear.
An intervention study has established human gut microbes associated with obesity \citep{ley2006microbial}.
An obese phenotype can be transferred from mouse to mouse by gut microbiome transplantation \citep{turnbaugh2006obesity}.
Pregnant human gut microbiome leads to obesity in mice \citep{koren2012host}.
However, a study of obesity in the old-order Amish did not find any correlation between obesity and particular gut communities \citep{zupancic2012analysis}.

Bacteria have been the primary focus of human microbiome research, and other domains have been investigated though to a lesser extent.
Changes in archaeal and fungi have been shown to covary with bacterial residents \citep{hoffmann2013archaea}.
Viral populations have been observed to be highly dynamic and variable across individuals \citep{reyes2010viruses,minot2011human,minot2013rapid}.

In this paper we will be primarily be describing the human microbiome from the perspective of ecological dynamics.
We will be ignoring recent research on fine-scale immune-mediated interactions between host and microbe \citep[reviewed in][]{hooper2012interactions}.
Our understanding of the true effect of the microbiome will eventually come from such a molecular-level understanding, although until we can characterize all of the molecular interactions between microbes and with the human body, ecological methods will continue to have an important role.


\section{Investigating the human microbiome via sequencing}
It is now possible to assay microbial communities in high throughput using via sequencing.
There are two ways of doing so.
The first is to amplify a specific ``marker'' gene in the genome, and perform sequencing.
The second is to randomly shear input DNA and/or RNA and then perform sequencing directly.
Although these words are not consistently used in the literature, we will consistently refer to the former as a \textit{survey} and the second a \textit{metagenome}.

Lots of survey, metagenome, and whole-genome sequencing data has been generated by the Human Microbiome Project \citep{methe2012framework} and is available on its website\footnote{\url{http://www.hmpdacc.org/}}.
The METAHIT study also generated lots of data but it is not freely available. NNN

\subsection{Microbial community estimation using marker gene surveys}
Our modern knowledge of the microbial world is due to a large part derived from the methods of Carl Woese and colleagues who pioneered the use of marker genes as a way to distinguish between microbial lineages \citep{fox1977comparative}.
Their work, and the scientists who followed them, used especially the 16S ribosomal gene.
This gene was chosen because it has regions of high and low diversity, which enable resolution on a variety of evolutionary time scales.
Regions of low diversity in 16S also enabled the development of the first ``universal'' 16S PCR primers \citep{lane1985rapid} which initiated surveys of all organisms regardless of whether they can be cultured.

Where Woese and colleagues labored over digestion and gel electrophoresis to infer sequences, modern researchers have the luxury of high throughput sequencing.
This can be done with a high level of multiplexing, making an explicit trade-off between depth of sequencing for each specimen and the number of specimens able to be put on the sequencer at the same time.
This has led to extensive parallelization, most recently by sequencing dozens of samples on the Illumina instrument \citep{degnan2011illumina,caporaso2012ultra}.
This leads to the question of how many sequences are needed to characterize the microbial diversity of a given environment.
For simply separating two rather different samples, relatively few sequences per sample are required \citep{kuczynski2010microbial} however for more subtle information deeper sequencing is required.

Despite the high-throughput and low cost of modern sequencing, inherent challenges remain for the use of population census by marker gene sequencing.
Most fundamentally, various microbes have different DNA extraction efficiencies, even with hard core protocols, meaning that the representation of marker gene sequences is not representative of the actual communities \citep{morgan2010metagenomic}.
Current sequencing technology is limited to a length that is shorter than most genes, which limits the resolution of the analyses.
``Primer bias,'' or differing amplification levels of various sequences based on their affinity for the primers \citep{suzuki1996bias,polz1998bias}, is a challenge and has led to the standardization of primers \citep{methe2012framework}.
Worse, multiplex PCR is known to create chimeric sequences via partial PCR products \citep{hugenholtz2003chimeric,ashelford2005least,haas2011chimeric,schloss2011reducing}, and chimera checking software has been developed \citep[including][]{ashelford2006new,edgar2011uchime}.
Also, 16S can be present in up to 15 copies and there can be diversity within the copies \citep{klappenbach2001rrndb};
recent work \citep{kembel2012incorporating} implements the independent contrasts \citep{felsenstein1985phylogenies} method to correct for copy number, which has been successful despite a moderate evolutionary signal \citep{klappenbach2000rrna}.
Some groups have reported advantages to using alternate single-copy genes as markers for characterization of microbial communities \citep[e.g.][]{case2007rpob,mcnabb2004hsp65}, however 16S remains the dominant locus used by a large margin.

XXX Data through time.

\subsection{Metagenomes}
As described above ``metagenome" means that data is sheared randomly across the genome, thus the genetic region of a read is unknown in addition to the organism it was derived from.
Because it does proceed through an amplification step, it does not have the same primer biases as a marker gene survey, although sequencing is known to have biases.

It is possible to use metagenomic data as an expanded set of marker genes.
That is, one can use 16S reads that appear in the metagenome as well as other ``core" genes present in a large proportion of micro-organisms that are expected to follow the same evolutionary path \citep{von2007quantitative,wu2008amphora,stark2010mltreemap,kembel2011phylogenetic}.
Because of the diversity of gene repertoire in microbes, the gene sets may have only limited overlap with one another, and even the largest collection of genes in these databases only recruits around 1 percent of a metagenome.

Metagenomic data can be used to infer information about metabolic capacity \cite{greenblum2012metagenomic,abubucker2012metabolic}.
The authors of the metAMOS pipeline \citep{treangen2013metamos} report speedups and much higher accuracy when reads are assembled before they are classified.


\subsection{Whole genomes}
Whole-genome sequencing from culture is currently being used for microbial outbreak tracking \citep{koser2012rapid,snitkin2012tracking}.
The Food and Drug Administration maintains GenomeTrakr, an openly accessible database of whole genome sequences from culture\footnote{\url{http://www.fda.gov/Food/FoodScienceResearch/WholeGenomeSequencingProgramWGS/}}.
This data will become common for unculturable organisms as single-cell sequencing \citep[reviewed in][]{kalisky2011single} becomes common.
The assembly of complete genomes from metagenomes, once limited to samples with a very small number of organisms \citep{baker2010enigmatic}, is now becoming feasible for more diverse populations with improved sequencing technology and computational approaches \cite{howe2012assembling,pell2012scaling,iverson2012untangling,emerson2012metagenomic,podell2013assembly}.


\section{Tree-thinking in human microbiome research}

In this section I consider the ways in which phylogenetic thinking has impacted human microbiome research.
What may be most interesting for this audience is the way in which phylogenetic trees are being used to actively revise taxonomy and used as a structure on which to perform sample comparison.

\subsection{Phylogenetics and taxonomy}

Phylogenetic inference has had a substantial impact on microbial ecology research by changing our view of the taxonomic relationships between microorganisms.
The most stunning example of this is the discovery that archaea, although morphologically similar to bacteria, form their own separate lineage \citep{woese1977phylogenetic}.

In general, this sort of approach has led to a number projects that use phylogeny to revise taxonomy.
These attempts are less ambitious than the development of the PhyloCode \citep{forey2001phylocode}, and simply work to revise the hierarchical structure of the taxonomy while (for the most part) leaving taxonomic names fixed to preserve information associated with taxonomic classifications.
Bergey's Manual of Systematic Bacteriology has officially adopted 16S as the basis for their taxonomy, although the actual revision process is somewhat opaque \citep{kreig1984bergey}.
The GreenGenes taxonomy \citep{desantis2006greengenes} has been very active in updating their taxonomy according to 16S, first with their GRUNT tool \citep{dalevi2007automated} and more recently with their tax2tree tool \citep{mcdonald2011improved} which uses a heuristic algorithm to optimize $F$-measure of precision and recall.
Interestingly, this latter tool allows for polyphyletic taxonomic groups as allowance for either lack of resolution of the 16S gene or taxonomic groups with no evolutionary basis.
Our group \citep{matsen2012reconciling} has developed ways of quantifying discordance between phylogeny and taxonomy based on a previously described coloring problem.

An obvious application of phylogenetics is to perform taxonomic classification, as the taxonomy is at least in part defined by phylogeny.
There is also SAP.

However, comparisons of taxonomic classification programs \citep{liu2008accurate,bazinet2012comparative} have indicated that current implementations of phylogenetic methods do not perform as well as simple na\"ive Bayes classifiers based on $k$-mer composition \citep{wang2007naive,rosen2008metagenome}.
% \citep{werner2011impact} Impact of training sets on classification of high-throughput bacterial 16s rRNA gene surveys
Some authors report that a combination of composition-based and homology-based classifiers work best \citep{brady2009phymm,parks2011classifying}.

A fascinating approach is that of \citet{segata2012metagenomic}, who start by compiling a database of clade-specific genes \citep{segata2011metagenomic}.
Then they classify the origin of a given read as being the only clade that has this gene.
They show that this has good specificity, however, sensitivity is limited to genomes that have been sequenced.

\subsection{The role of Operational Taxonomic Units (OTUs)}
There continues to be a lively debate on if there is a meaningful concept of species for microbes NNN, however most microbial research has replaced any traditional species concept with the notion of an Operational Taxonomic Units (OTUs).
An OTU is a proxy species concept that is typically defined with a fixed divergence cutoff, most commonly a 97\% sequence identity, such that each OTU consists of sequences that are closer to each other than that cutoff.
It is common for trees to be built on sequence representatives from these OTUs, and the abundance of an OTU to be given by the number of sequences that sit within that cluster.

There is an entire industry of OTU inference software with various strategies of various speeds; we will not attempt a systematic review here.
Of the fixed-cutoff methods, the traditional choice has been CD-HIT \citep{li2006cdhit}, which seems to have been supplanted by clustering features of USEARCH \citep{edgar2010usearch}.
These methods are very fast and heuristic, in that they are described as an algorithm rather than a routine attempting to optimize some notion of goodness of clustering.
It is well known that different ways of doing this heuristic clustering can result in very different results \citep{white2010alignment}.
There are also methods that try to come up with sequence groupings that more reflect the type of species divisions found in taxonomies \citep{navlakha2009finding}.
The centrality of the OTU concept can be seen by the fact that the contingency table of OTU observations is considered to be the fundamental data type NNN, or that methods have been devised to find OTUs from non-overlapping sequences NNN.
With the exception of this last reference, OTU inference is not considered to be a phylogenetic problem but rather something to be performed before phylogenetic inference begins.


\subsection{Diversity estimates using phylogenetics}
Because 16s surveys are inherently complex and noisy data, summary statistics are often used; summaries of the diversity of a single sample are often called \emph{alpha diversity}.
For the most part, this literature adapts methods from the classical ecological literature replacing taxonomic groupings with OTUs.
%The most common diversity indices are Simpson and Shannon.
% \citep{hill1973diversity} Diversity and evenness: a unifying notation and its consequences
However, phylogenetic diversity metrics are also used and here we will focus on their applications and methods.

Despite the enthusiasm with which microbial ecologists have accepted between-community comparison using phylogenetics (next section), phylogenetic alpha diversity seems under-developed.
Phylogenetic diversity notions have been applied on metagenomic reads expanded marker tree \citep{kembel2011phylogenetic}.
Although abundance-weighted non-phylogenetic diversity measures such as NNN simpson NNN shannon are among the most common, abundance-weighted phylogenetic diversity measures are generally not used.
Such measures do exist \citep{chao2010phylogenetic}
We have recently shown that partially-weighted abundance diversity measures do a good job of distinguishing between dysbiotic and "normal" states \citep{mccoy2013abundance}, in particular that they do a better job than the traditionally-used OTU-based measures.
We have also looked at the mean and variance of PD under random subsampling \citep{nipperess2013mean}.


\subsection{Community comparison using phylogenetics}
The level of similarity between samples or groups thereof called \emph{beta diversity}.
As with alpha diversity, it is common to use classical measures \citep[e.g.][]{jaccard1908nouvelles} applied to OTU counts, however in this case phylogenetics-based methods are widely used.
They generally go under variants of ``UniFrac'' as named by Lozupone and Knight \citep{LozuponeKnightUniFrac05;LozuponeEaWeightedUnifrac07}.
As evidence of their superiority
\citep{kuczynski2010microbial} Microbial community resemblance methods differ in their ability to detect biologically relevant patterns

The name UniFrac is a contraction of ``Unique Fraction,'' which refers to the fact that the original UniFrac definition compares the fraction of edges that connect only tips from one sample or another rather than between samples.
Weighted UniFrac is an abundance weighted version.
We have shown that weighted UniFrac is in fact a special case of the earth-movers distance, and is special in that it can be calculated in linear time \citep{evans2012phylogenetic}.
We also showed that the commonly-used randomization procedure to attach a p-value to an observed distance has a central limit theorem approximation as a Gaussian process.
Recent work \citep{chen2012associating} has shown that a partially abundance-weighted variant of UniFrac has greater power to resolve community differences than either unweighted or weighted UniFrac.

The most common way to use a pairwise distance matrix found from an application of UniFrac is to apply an ordination method such as principal coordinates analysis.
It recovers gradients.
\citep{nemergut2011global}
?? Global patterns in the biogeography of bacterial taxa
Indeed, the observation of differences between two communities is often used as prima fasce evidence of a difference between them, while the lack of such a difference indicates that the communities are not different (amish?)

Figure: example principal coordinates plot.

There have been efforts to augment the ordination visualizations with additional information giving more structure to the visualizations.
Biplot.
Elizabeth Purdom \citep{BikEaMicrobiotaStomach06,PurdomAnalyzingDataGraphs08} has incorporated.
We have developed an alternative ordination process that explicitly labels the axes with weightings on phylogenetic trees that indicate their meanings \citep{matsen2013edge}.


\subsection{Phylogeny and function}

16S distance is frequently used as a proxy for a functional comparison.
Those accustomed to microbial genetics may think this surprising, because the genetic repertoire of microbes is commonly acquired horizontally as well as vertically, and horizontal transmission may leave no trace in the 16S ancestry.

However, \citep{zaneveld2010ribosomal} have shown that organisms that are more distant in terms of 16S are also more divergent in terms of gene repertoire.
Such observations generally follow a nonlinear curve, and the extent to which they lie on the curve appears to be phylum-dependent.
This approach has recently been taken to its logical conclusion by trying to infer functional characteristics using discrete trait evolution models on 16S gene trees \citep{langille2013predictive} by either parsimony \citep{kluge1969quantitative} or likelihood \citep{pagel1994detecting} methods via the ape package \citep{paradis2004ape}.

Similar logic has been applied to prioritize microbes for sequencing.
\citet{wu2009phylogeny} have derived a ``phylogeny-driven genomic encyclopaedia of Bacteria and Archaea'' by selecting organisms for sequencing that are divergent from organisms that have a sequenced genome.
By doing so they have recovered more novel open reading frames than they would have using other methods NNN.
However, for the equivalent effort for the human microbiome \cite{fodor2012most} list phylogenetic methods were not used-- they just wanted to get microbes that were prevalent in the microbiome but not sequenced.
NNN wasn't there a previous effort that used ppacer?
% http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0041294


\subsection{Genome-scale inquiries using phylogenetics}
With some notable exceptions, mainstream applications of phylogenetics have typically been with the idea of finding ``the'' tree of a collection of human-associated microbes rather than explicitly exploring divergence between various gene trees.
As described above, whole-genome data is typically used to directly infer functional information rather than information concerning ancestry.
The debate concerning whether a microbial tree of life is a useful concept does not seem to have dampened human microbiome researchers' enthusiasm for using such a tree.

Nevertheless, the work that has been done to infer horizontal gene transfer has revealed interesting results.
\citet{hehemann2010transfer} found that a gene found in seaweed has been transferred into the microbiome of Japanese such that individuals with this resulting microbiome are better able to digest algae.
Following on this work, \citet{smillie2011ecology} found that the human microbiome is in fact a common location for gene transfer.

NNN look for gene transfer work in microbiome.


\section{Phylogenetic inference as practiced by human microbiome researchers}

\subsection{Alignment and tree inference}
In general, human microbiome researchers are interested in quickly doing phylogenetic inference on large data sets, and are less interested in clade-level accuracy or measures of uncertainty.
This can be defended by saying for many applications such as UniFrac, the tree is used as a framework to structure the data, and there is a certain amount of flexibility in that framework that will give the same results.
Furthermore, given that the underlying data is typically 16S only we can expect some tree problems even with the best methods.
Additionally, as specified below, these data sets can be very large.
There does not seem to be contentious discussion of specific features of the inferred trees equivalent to, say, the current discussion around the rooting of the placental mammal tree.

Alignment is focused on creation of large curated alignments, and alignment of additional sequences to those established alignments.
There are two primary such projects: one is the SILVA project \citep{pruesse2007silva,quast2013silva}, and the other is the GreenGenes \citep{desantis2006greengenes,mcdonald2011improved} database.
Because of the high rate of insertion and deletion of nucleotides in 16S, these alignments have a high percentage of gap.
Taking the length of 16S to be 1543 nt,
% SILVA
% 1543 / 42284
% .96353230536373096207
SILVA alignment version 115 has XXX sequences and is over 96\% gap,
% GreenGenes
% 1543 / 7682
% .79914084873730799272
while the GreenGenes 13\_5 alignment has 1,262,986 sequences and  is almost 80\% gap when aligned by SSU-Align \cite{nawrocki2009structural}
The SILVA alignment is... hand aligned?

Such large data

MSA tool specializing in 16S \cite{pruesse2012sina}.

PYNAST
ARB alignment.

\citep{eddy1998profile}
Profile hidden Markov models.

\citep{nawrocki2009infernal}
Infernal 1.0: inference of RNA alignments

\cite{peplies2008standard,yarza2008all}


\citep{felsenstein1981evolutionary}
{Evolutionary trees from DNA sequences: a maximum likelihood approach}

NJ
\citep{price2010fasttree}
{FastTree 2--approximately maximum-likelihood trees for large alignments}

Living tree project.

Bayesian phylogenetic inference is absent.

However, it is surprising that so much focus is on complete de novo tree inference, and that methods such as DCM NNN have not gained traction.

Networks not considered.

The scale of the data has motivated strategies other than complete phylogenetic inference.
Although such insertion has a long history as a means to sequentially build a phylogenetic tree NNN farris
, the first software with insertion specifically as a goal was by \citep{ludwig2004arb} which works by parsimony.
ARB is used to reconstruct a full tree.

There are also other methods with the less ambitious goal of mapping sequences of unknown origin into a tree.
Other analogous methods have been used \citep{wu2008simple,monierEaLargeViruses08,vonMeringEaQuantitative08,stark2010mltreemap,matsen2010pplacer,berger2011performance} with various speeds and features.
Such questions have also found the interest of the alignment community as well.
\citet{berger2011aligning} focus on the problem of inferring the optimal alignment and insertion of sequences into a tree.
\citet{mirarabsepp} does an approximately.

There are other strategies still that do something related to phylogenetics.
\citep{huson2007megan}
LCA strategy


\subsection{Software}

People do most phylogenetic inferences as part of a pipeline.
The most popular ones are QIIME \citep{caporaso2010qiime} and mothur \citep{schloss2009introducing}.
AXIOME streamlines and manages analysis of small subunit (SSU) rRNA marker data in QIIME and mothur \citep{lynch2013axiome}.

mothur has ported in the clearcut \citep{evans2006relaxed,sheneman2006clearcut} program.
QIIME wraps clearcut and \citep{price2010fasttree}.

Databases:
\citep{chen2010human}
The Human Oral Microbiome Database: a web accessible resource for investigating oral microbe taxonomic and genomic information

\citep{griffen2011core}
CORE: a phylogenetically-curated 16S rDNA database of the core oral microbiome

\citep{srinivasan2012bacterial}


\section{Phylogenetic challenges and opportunities in human microbiome research}

In addition to the usual challenges surrounding large-scale phylogenetic inference, there are some questions that are specific to the microbial setting.
I have collected some of these questions here.

In this paper I have devoted considerable space to the ways in which microbial ecologists have used the 16s tree as a proxy structure for the evolutionary history of their favorite organisms.
They have even shown that 16S distance recapitulates gene content divergence and used this correlation to predict gene functions.
16s tree has taken us remarkably far, but it is not a complete representation of the evolutionary history of these microbes.

The apparent success of 16s tree based comparisons begs the question of if a more complete representation of the evolutionary history of the microbes would yield better predictions.
This yields a practical perspective on the theoretical issue of the tree of life: what is the representation of the genetic ancestry of an organism that allows us to best predict features of underlying genomes?
This representation could be simple.
For example, one of the results of \citet{zaneveld2010ribosomal} is that 16S correlates better with gene repertoire in some taxonomic groups than others.
If we were to equip the 16S tree with some measure of the strength of that correlation, would that allow for more accurate prediction?
Would a species tree reconciled with a collection of gene trees allow for better prediction?
It is quite possible that there is nothing better given the inherent noise of the data, but if we allow an arbitrary ``hidden'' object, what such object would perform best?
For example, collections of reconciled gene trees in the presence of gene deletion, transfer, and loss, and could be used \citep[see][for interesting recent results]{szollHosi2013lateral,szollHosi2013efficient}.

In addition to using phylogenies to infer gene repertoire, it appears that neutral models involving phylogenetics could be more fully developed.
Methods explicitly invoking trait evolution seem notably absent, with one recent exception \cite{langille2013predictive}.
The results of this simple method are reasonable.
It is possible that improved methods, perhaps involving whole-genome evolutionary modeling or models of metabolic network evolution, could shed light on the problem.
How could we formulate a useful notion of independent contrasts \citep{felsenstein1985phylogenies} on a collection of reconciled gene trees?

Given that microbes are cosmopolitan and have been evolving for a long time, another important project is to model community assembly.
One direction is to apply Hubbell's neutral theory to the human microbiome \citep{fierer2012animalcules,costello2012application}.
A recent paper does so with an explicitly phylogenetic perspective \citep{o2012phylogenetic}, and includes some comparison of models to data.
Continued work in this direction seems warranted, given the way in which phylogenetic tree shape statistics have had a significant impact on macroevolutionary modeling \cite{mooers1997inferring,aldous2011five}.
In this case various (alpha and beta) diversity statistics would take the role of tree shape statistics.
It will be challenging to bring together macroevolutionary modeling with species abundance modeling, but some very first steps have been made in another setting \citep{lambert2013predicting}.

Diversity preservation is of interest for microbiome researchers, but has not received the formalization and algorithmic treatment surrounding phylogenetic diversity for larger organisms.
Martin Blaser in particular has argued that changes in our microbiomes are leading to an increase in autoimmune disease and certain types of cancer and has made passionate appeals to preserve microbiome diversity \cite{blaser2011antibiotic,cho2012human}.
Indeed, studies show that the modern lifestyle, including antibiotic use, has led to significant changes in microbiome.
Because microbiomes are seeded from mother to child, there is a somewhat equivalent notion of extinction of microbiome when the chain is interrupted.
Consequently, a recent study has explicitly contrasted microbiome development in urban, forest-dwelling, and rural African populations \cite{yatsunenko2012human}, and one study has endeavored to characterize the microbiome from ancient feces \cite{tito2012insights}.

Perhaps in part because of the importance of mother-to-child transmission, there are indications of coevolution between microbiomes and their hosts.
\citet{ochman2010evolutionary} found identical trees for primate evolution and microbiome,
They used maximum parsimony such that each column was a microbe and in each such entry took discrete states according to how much of that microbe was present.
Although parsimony gave an interesting answer here, the presence of such coevolution raises the question of what sort of forward-time models are appropriate for microbiome change?
Would methods using these models do better than clustering or distance methods applied to the above distances?
Other studies \citep[e.g.][]{phillips2012microbiome,delsuc2013convergence} see a combination of historical and dietary influences.
This seems like a setting in which convergent evolution could be quantified.

Finally, the conventional wisdom from the microbiome community that conclusions of a UniFrac analysis are robust to tree reconstruction methodology begs further exploration.
Would it be possible to infer an ``equivalence class'' of phylogenetic trees, where two trees are deemed equivalent if they induce the same principal coordinates projection with the same underlying data?
Given that a tree is an integral part of a UniFrac analysis, it would also be interesting to be able to infer the features of a tree that determine the primary trends in a projection.
% How bad of tree inference can we do and still get the same trends?


\section{Discussion}
What we can expect next.

Future will give deeper sequencing, which can mean massive parallelization.
Citizen microbiome projects.

% Companies? http://www.ribocon.com/ second genome, etc

Comparative studies of microbiomes.
Microbiome analysis among bats describes influences of host phylogeny, life history, physiology and geography

Clinical applications,
\citep{clarridge2004}
{{I}mpact of 16{S} r{RNA} gene sequence analysis for identification of bacteria on clinical microbiology and infectious diseases}

For routine identification of a single microbe in culture, MALDI-TOF kicks ass and is cheap.
Diagnostics involving mixed communities.

Bioinformatics will get hardened into fixed workflows.

Understand cause and effect of diseases.
Take an ecological view on atopic dermititis.

Although we have raised the possibility of using organism-level modeling via phylogeny
There is the possibility that we don't have to think about individual organisms at all, but rather can think about the collection of genes and their metabolic network as a meta-organism (cite Elbo).
Has limitation that you forget about cellular boundaries, which matter-- not a freely diffusing soup.
Question: to what extent is modeling cellular-level evolutionary history interesting?
As an extension microbes have to be close to one another-- requires population modeling.

There are limitations to what we can learn using genetics because more intricate processes such as gene regulation may be at play, limiting what sequence-level phylogenetics can do.
However, evolution still acts and there is nothing saying that we couldn't model evolution of phenotype in some other way.

Future work will need to include specific interactions.
Host side is sometimes missing, but experiments are good.
As described above, we can expect much more development in terms of specifics.
\citep{hooper2012interactions}

Although I have described some areas for future research for phylogeneticists, researchers should be warned that their work may not be widely accepted by the microbial ecology community.
That is, methodological changes that would be viewed as improvements in, substitution modeling for example, may not readily be taken up by the microbial ecology community.
My sense is that this is in part because the bulk of microbial ecology researchers are doctors, immunologists, and oceanographers that have an applied focus and for which computational and modeling issues are a nuisance rather than an interesting aspect.
For this reason, computational analysis is primarily done in advance or as part of established pipelines and is hardened into standard operating procedures \citep{peplies2008standard}.

Human microbiome research has experienced a frenetic rate of expansion over the past decade, and sometimes it can feel like researchers in this area have forgotten some of the basic principles of study design, such as power calculations, that are well recognized in other areas of medical research.
Although the human microbiome has gained a lot of attention recently, and in some cases too much attention, our microbes are here to stay and so is research on them.
Thus we can look forward to the field of human microbiome analysis settling down to a comfortable and mature middle age as an interesting intersection between ecology and medicine.
Phylogenetics has contributed significantly to ecology in the past, and indeed has already contributed significantly to microbial ecology; the next generation of methods may require integration of genome-level data.



\section{Acknowledgements}
Aaron Darling, Connor McCoy,
FHCRC folk.
ATD grant.


\notforarxiv{
\newpage
\section{Figure Legends}
%\FIGmassTransport
\clearpage

\newpage
}

\bibliographystyle{plainnat}
\bibliography{sbreview}

\end{document}


