
\documentclass{amsart}
\usepackage{amsmath,amsfonts,amssymb,amsthm}
\usepackage[english]{babel}
\usepackage{graphicx}
\usepackage{url}
\usepackage{palatino}
\usepackage[round]{natbib}
\newcommand{\forarxiv}[1]{#1}
\newcommand{\notforarxiv}[1]{}

% show keys for eqs, etc.
% \usepackage[notref,notcite]{showkeys}

% code
\newcommand{\program}{\textsf{program}}

\newcommand{\eat}[1]{}

\newcommand{\FIGmassTransport}{\
\begin{figure}[ht]
\begin{center}
  \forarxiv{\includegraphics[width=13cm]{mass_transport.pdf}}
\end{center}
\caption{\
  Caption goes here.
}
\label{FIGmassTransport}
\end{figure}
}
\newcommand{\refFIGmassTransport}{1}

\hyphenation{Ge-nome Ge-nomes Me-ta-ge-nome Me-ta-ge-nomes Ma-cro-ev-o-lu-tion-ary}

% Noah suggests making a table for
%alpha diversity
%beta diversity
%PCA
%taxonomic assignment/classification
%clustering/binning

\begin{document}

\notforarxiv{
\begin{flushright}
Version dated: \today
\end{flushright}
\bigskip
\noindent RH: PHYLOGENETICS AND THE HUMAN MICROBIOME
\bigskip
\medskip
\begin{center}

\noindent{\Large \bf Phylogenetics and the human microbiome.}
\bigskip

\noindent {\normalsize \sc
Frederick A. Matsen IV$^1$}\\
\noindent {\small \it
$^1$
Program in Computational Biology, Fred Hutchinson Cancer Research Center, Seattle, WA, 91802, USA}\\
\end{center}
\medskip
\noindent{\bf Corresponding author:} Frederick A Matsen, Program in Computational Biology, Fred Hutchinson Cancer Research Center, Seattle, WA, 91802, USA; E-mail: matsen@fhcrc.org.\\
\vspace{1in}
}

\forarxiv{\
\title{Phylogenetics and the human microbiome.}
\author{Frederick A. Matsen IV}
\date{\today}
\begin{abstract}
}
\notforarxiv{
\subsubsection{Abstract}
}

The human microbiome is the collection of microbes that live inside and on the outside of humans.
Recent work including the Human Microbiome Project (HMP) and the Metagenomics of the Human Intestinal Tract (MetaHIT) consortia has advanced human microboime research by generating thousands of 16s surveys and terabases of metagenomic data on the human microbiome.
In this paper I review the impact of phylogenetics and tree-thinking on the methods used in the analysis of human microbiome data, as well as describing current challenges for phylogenetics coming from this type of work.

\forarxiv{
\end{abstract}
\maketitle
}

\notforarxiv{
\noindent (Keywords: human microbiome; microbial ecology; phylogenetic methods; review)\\
\vspace{1.5in}
}


\section{Introduction}

The parameter regime and focus of microbiome research sits outside of the traditional setting for systematic biology, thus why should our community be interested in what these microbial ecologists and medical researchers have done?
This system is data- and question-rich.
It absolutely requires molecular methods, as microbes can be identified by their molecular sequence, which is much more straightforward to do in high throughput than morphological or phenotypic characterization.
Indeed, just to do ecology you have to do sequencing.
Many of the issues that are raised by high throughput sequencing of beasties that are more commonly of interest for phylogenetics folk are present, and worse, in microbial studies.
For instance, the diversity of these organisms is great and organizing them into a taxonomy is a serious challenge, especially in the absence of obvious morphological features.

Because of challenges with species definitions, microbial ecology researchers have developed their own techniques for doing comparing samples which are explicitly phylogenetic.
Although there is some overlap with previous literature, these techniques are novel and could be used in a wider setting and deserve wider consideration by the phylogenetics community.

The human microbiome part of microbial ecology is specifically interesting because questions of microbial genomics, translated into questions of function, have important consequences for human health.
Additionally, due to more than a century of hospital lab medicine, our knowledge about human-associated microbes is much richer than about microbes in general.
It is relatively routine to manipulate the human microbiome or models thereof via intervention studies and germ-free animals.

In this review I will describe human microbiome research from the perspective of a phylogenetics researcher.
I will first briefly review the recent literature on the human microbiome, then describe novel ways in which human microbiome researchers have used trees and the challenges that human microbiome research poses for the phylogenetics community.
I will finish with opportunities for the \textit{Systematic Biology} audience to contribute to this field.

\section{The human microbiome}
The human microbiome is the collection of microbial organisms that live inside of and on the surface of humans.
These organisms are populous: it has been estimated that there are ten times as many bacteria associated with each individual than there are human cells.
The microbiome has remarkable metabolic potential, with a set of genes estimated to be about 150 times larger than the human collection of genes \citep{qin2010human}.
Much of our metabolic interaction with the outside world is mediated by our microbiome, as it has important roles in both vitamin
% http://www.sciencedirect.com/science/article/pii/S095816691200119X
and drug \citep{maurice2013xenobiotics} metabolism; our food and drug intake also impacts the diversity of microbes present.
In this section I will briefly review what is known about the human microbiome and its effect on our health.

The human microbiome is an ecosystem.
The microbiome is dynamic in terms of representation but apparently constant in terms of function \citep{hmp2012structure}.
The ``core'' microbiome, or the microbiome shared between all humans, has been described \citep{turnbaugh2008core}.
Human microbiome is spatially organized, as can be seen on skin \citep{grice2009topographical}, with substantial variation in human body habitats across space and time \citep{costello2009bacterial}.
There is a substantial range of inter-individual versus intra-individual variation \citep{hmp2012structure}.

Our actions can shift the composition of our microbiome.
Changes in diet very quickly shift its composition, and there is a strong correlation between long-term diet and microbiome \citep{li2009human,wu2011linking}.
Antibiotics fundamentally disturb microbial communities, resulting in an effect that lasts for years \citep{jernberg2007long,dethlefsen2008pervasive,jakobsson2010short,dethlefsen2011incomplete}.

Considerable attention has been given to how the microbiome influences host phenotype.
There is strong evidence of interaction between the gut microbiome and obesity, but the story is yet clear.
An intervention study has established human gut microbes associated with obesity \citep{ley2006microbial}.
An obese phenotype can be transferred from mouse to mouse by gut microbiome transplantation \citep{turnbaugh2006obesity}.
Pregnant human gut microbiome leads to obesity in mice \citep{koren2012host}.
However, a study of obesity in the old-order Amish did not find any correlation between obesity and particular gut communities \citep{zupancic2012analysis}.

Bacteria have been the primary focus of human microbiome research, however other domains have been investigated.
Archaea and fungi interact with bacterial residents \citep{hoffmann2013archaea}.
Virus populations have been observed to be highly dynamic and variable across individuals \citep{reyes2010viruses,minot2011human,minot2013rapid}.

In this paper we will be primarily be describing the human microbiome from the perspective of ecological dynamics.
We will be ignoring recent research on fine-scale immune-mediated interactions between host and microbe \citep[reviewed in][]{hooper2012interactions}.
Our understanding of the true effect of the microbiome will eventually come from such a molecular-level understanding, although until we can characterize all of the molecular interactions between microbes and with the human body, ecological methods will continue to have an important role.


\section{Investigating the human microbiome via sequencing}
It is now possible to assay microbial communities in high throughput using via sequencing.
There are two ways of doing so.
The first is to amplify a specific ``marker'' gene in the genome, and perform sequencing.
The second is to randomly shear input DNA and/or RNA and then perform sequencing directly.
Although there is some vagueness in the usage of these words, we will consistently refer to the former as a \textit{survey} and the second a \textit{metagenome}.

Lots of survey, metagenome, and whole-genome sequencing data has been generated by the Human Microbiome Project \citep{methe2012framework} and is available on its website\footnote{\url{http://www.hmpdacc.org/}}.

\subsection{Microbial community estimation using marker gene surveys}
Our modern knowledge of the microbial world is due to a large part derived from the methods of Carl Woese and colleagues who pioneered the use of marker genes as a way to distinguish between microbial lineages \citep{fox1977comparative}.
Their work, and the scientists who followed them, used especially the 16S ribosomal gene.
This gene was chosen because it has regions of high and low diversity, which enable resolution on a variety of evolutionary time scales.
Regions of low diversity in 16S also enabled the development of the first ``universal'' 16S PCR primers \citep{lane1985rapid} which initiated surveys of all organisms regardless of whether they can be cultured.

Where Woese and colleagues labored over digestion and gel electrophoresis to infer sequences, modern researchers have the luxury of high throughput sequencing.
This can be done with a high level of multiplexing, making an explicit trade-off between depth of sequencing for each specimen and the number of specimens able to be put on the sequencer at the same time.
This has led to extensive parallelization, most recently by sequencing dozens of samples on the Illumina instrument \citep{degnan2011illumina,caporaso2012ultra}.
This leads to the question of how many sequences are needed to characterize the microbial diversity of a given environment.
For simply separating two rather different samples, relatively few sequences per sample are required \citep{kuczynski2010microbial} however for more subtle information deeper sequencing is required.

Despite the high-throughput and low cost of modern sequencing, inherent challenges remain for the use of population census by marker gene sequencing.
Most fundamentally, various microbes have different DNA extraction efficiencies, even with hard core protocols, meaning that the representation of marker gene sequences is not representative of the actual communities \citep{morgan2010metagenomic}.
Current sequencing technology is limited to a length that is shorter than most genes, which limits the resolution of the analyses.
``Primer bias,'' or differing amplification levels of various sequences based on their affinity for the primers \citep{suzuki1996bias,polz1998bias}, is a challenge and has led to the standardization of primers \citep{methe2012framework}.
Worse, multiplex PCR is known to create chimeric sequences via partial PCR products \citep{hugenholtz2003chimeric,ashelford2005least,haas2011chimeric,schloss2011reducing}.
Also, 16S can be present in up to 15 copies and there can be diversity within the copies \citep{klappenbach2001rrndb};
recent work \citep{kembel2012incorporating} implements the independent contrasts \citep{felsenstein1985phylogenies} method to correct for copy number, which has been successful despite a moderate evolutionary signal \citep{klappenbach2000rrna}.
Some groups have reported advantages to using alternate single-copy genes as markers for characterization of microbial communities \citep[e.g.][]{case2007rpob,mcnabb2004hsp65}, however 16S remains the dominant locus used by a large margin.


\subsection{Metagenomes}

As described above ``metagenome" means that data is sheared randomly.
This type of data has a complication above that taken from 16S survey data, which is that the genetic region sequenced is unknown in addition to the organism it was taken from.
However, because it does proceed through an amplification step, it does not have the same primer biases as a marker gene survey, although sequencing is known to have biases.

It is possible to use metagenomic data as an expanded set of marker genes.
That is, one can use 16S reads that appear in the metagenome as well as other ``core" genes present in a large proportion of micro-organisms that are expected to follow the same evolutionary path \citep{von2007quantitative,wu2008amphora,stark2010mltreemap,kembel2011phylogenetic}.
Because of the diversity of gene repertoire in microbes, the gene sets may have only limited overlap with one another, and even the largest collection of genes in these databases only recruits around 1 percent of a metagenome.

Metagenomic data can be used to infer information about metabolic capacity \cite{greenblum2012metagenomic,abubucker2012metabolic}.
The authors of the metAMOS pipeline \citep{treangen2013metamos} report speedups and much higher accuracy when reads are assembled before they are classified.
The assembly of complete genomes from metagenomes, once limited to samples with a very small number of organisms \citep{baker2010enigmatic}, is now becoming feasible for more diverse populations with improved sequencing technology and computational approaches \cite{howe2012assembling,pell2012scaling,iverson2012untangling,emerson2012metagenomic,podell2013assembly}.


\subsection{Whole genomes}
Whole-genome sequencing from culture is currently being used for microbial outbreak tracking \citep{koser2012rapid,snitkin2012tracking}.
The Food and Drug Administration maintains GenomeTrakr, an openly accessible database of whole genome sequences from culture\footnote{\url{http://www.fda.gov/Food/FoodScienceResearch/WholeGenomeSequencingProgramWGS/}}.
This data will become common for unculturable organisms as single-cell sequencing \citep[reviewed in][]{kalisky2011single} becomes common.




\section{Tree-thinking in human microbiome research}

Phylogenetics is primarily used in this realm as a framework with which data can be organized.
It is common for researchers to equate 16S phylogeny with taxonomy.
UniFrac is interesting.

\subsection{Phylogenetics and taxonomy}

Phylogenetics has had a substantial impact on microbial ecology research by changing our view of the taxonomic relationships between microorganisms.
The most stunning example of this is the discovery that archaea, although morphologically similar to bacteria, form their own separate lineage \citep{woese1977phylogenetic}.

However, the reason people are attached to taxonomy is that there is a lot of information using taxonomic labels.

In general, this sort of approach has led to a number projects that use phylogeny to revise taxonomy.
These attempts are less ambitious than the development of the PhyloCode \citep{forey2001phylocode}, and simply work to revise the hierarchical structure of the taxonomy while (for the most part) leaving taxonomic names fixed.
Bergey's Manual of Systematic Bacteriology has adopted 16S \citep{kreig1984bergey}.
The GreenGenes taxonomy \citep{desantis2006greengenes} has been very active in updating their taxonomy according to 16S, first with their GRUNT tool \citep{dalevi2007automated} and more recently with their tax2tree tool \citep{mcdonald2011improved}.
Rather than proposing changes to the taxonomy, our group \citep{matsen2011reconciling} has developed ways of quantifying discordance between phylogeny and taxonomy.

Tree-based classification.
\citep{bazinet2012comparative}

A fascinating approach is that of \citet{segata2012metagenomic}, who start by compiling a database of clade-specific genes \citep{segata2011metagenomic}.
Then they classify the origin of a given read as being the only clade that has this gene.
They show that this has good specificity, however, sensitivity is limited to genomes that have been sequenced.


\subsection{Diversity estimates using phylogenetics}
Because 16s surveys are inherently complex and noisy data, summary statistics are often used; summaries of the diversity of a single sample are often called \emph{alpha diversity}.
For the most part, this literature adapts methods from the classical ecological literature.

Here we will focus on variants of phylogenetic diversity and their use.

\citep{hill1973diversity}
Diversity and evenness: a unifying notation and its consequences

On expanded marker tree \citep{kembel2011phylogenetic}.

hill numbers
\citep{chao2010phylogenetic}

\citep{odwyer2012phylogenetic}
Phylogenetic Diversity Theory Sheds Light on the Structure of Microbial Communities

Despite this work, and the enthusiasm with which microbial ecologists have accepted between-community comparison, phylogenetic alpha diversity seems under-developed.
We have recently shown that partially-weighted abundance diversity measures do a good job of distinguishing between dysbiotic and "normal" states \citep{mccoy2013abundance}.
We have also looked at PD under rarefaction \citep{nipperess2013mean}.


\subsection{Community comparison using phylogenetics}

The level of similarity between samples or groups thereof are often called \emph{beta diversity};
Traditional way: \citep{jaccard1908nouvelles} Nouvelles recherches sur la distribution florale

UniFrac distance and variants has been very successful.
\citep{LozuponeKnightUniFrac05}
\citep{LozuponeEaWeightedUnifrac07}
\citep{kuczynski2010microbial} Microbial community resemblance methods differ in their ability to detect biologically relevant patterns

It recovers gradients.
\citep{nemergut2011global}
?? Global patterns in the biogeography of bacterial taxa

Figure: example principal coordinates plot.

This has precedents.
\citep{evans2012phylogenetic}
The phylogenetic Kantorovich-Rubinstein metric for environmental sequence samples

\citep{BikEaMicrobiotaStomach06}
\citep{PurdomAnalyzingDataGraphs08}

\citep{chen2012associating}
Associating microbiome composition with environment-- generalized unifrac

\citep{matsen2013edge}
Edge principal components and squash clustering: using the special structure of phylogenetic placement data for sample comparison



\subsection{Phylogeny and function}

As we will see below, 16S is frequently used as a way of measuring diversity.
Those accustomed to microbial genetics may think this surprising, because the genetic repertoire of microbes may not be so much vertically inherited.
However, \citep{zaneveld2010ribosomal} have shown that there is a correlation between pairwise distances of 16S genes and genetic repertoire.
This approach has recently been taken to its logical conclusion by trying to infer functional characteristics using discrete trait evolution models on 16S gene trees \citep{langille2013predictive} using either parsimony \citep{kluge1969quantitative} or likelihood \citep{pagel1994detecting} methods via the ape package \citep{paradis2004ape}.


\subsection{Genome-scale inquiries using phylogenetics}

A phylogeny-driven genomic encyclopaedia of Bacteria and Archaea \citep{wu2009phylogeny}.

Most wanted.

Sushi paper, network of gene transfer.

Reconciliation, team phyldog.


\section{Phylogenetic inference as practiced by human microbiome researchers}

\subsection{Alignment and tree inference}

Very large data sets.

Some groups are not interested in accuracy per se.

Alignment.
\citep{eddy1998profile}
Profile hidden Markov models.

\citep{nawrocki2009infernal}
Infernal 1.0: inference of RNA alignments

PYNAST


\citep{felsenstein1981evolutionary}
{Evolutionary trees from DNA sequences: a maximum likelihood approach}

NJ
\citep{price2010fasttree}
{FastTree 2--approximately maximum-likelihood trees for large alignments}

Species delineation problem not explicitly considered. Clustering.
\citep{li2006cdhit}
Li, W.  and Godzik, A.
\citep{edgar2010usearch}
\citep{navlakha2009finding} Finding biologically accurate clusterings in hierarchical tree decompositions using the variation of information
\citep{white2010alignment} Alignment and clustering of phylogenetic markers-implications for microbial diversity studies

Networks not considered.

Chimeric sequences are a real problem, and similar to recombination.

ARB software.
Phylogenetic placement.

\citep{wu2008simple}
A simple, fast, and accurate method of phylogenomic inference

\citep{matsen2010pplacer}

\citep{ludwig2004arb}
{{ARB}: a software environment for sequence data}
\citep{berger2011performance}

\citep{berger2011aligning}
Aligning short reads to reference alignments and trees

\citep{mirarabsepp}
{SEPP: SAT{\'e}-Enabled Phylogenetic Placement}

\citep{monierEaLargeViruses08}
{Taxonomic distribution of large DNA viruses in the sea}

\citep{stark2010mltreemap}
{{MLTreeMap}-accurate Maximum Likelihood placement of environmental DNA sequences into taxonomic and functional reference phylogenies.}

\citep{vonMeringEaQuantitative08}
{Quantitative phylogenetic assessment of microbial communities in diverse environments}


\citep{huson2007megan}

\citep{wang2007naive}
{Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy}

\citep{werner2011impact}
Impact of training sets on classification of high-throughput bacterial 16s rRNA gene surveys



\subsection{Software}

People do most phylogenetic inferences as part of a pipeline.
The most popular ones are QIIME \citep{caporaso2010qiime} and mothur \citep{schloss2009introducing}.
AXIOME streamlines and manages analysis of small subunit (SSU) rRNA marker data in QIIME and mothur \citep{lynch2013axiome}.

mothur has ported in the clearcut \citep{evans2006relaxed,sheneman2006clearcut} program.
QIIME wraps clearcut and \citep{price2010fasttree}.

Bayesian inference is very rare.

Databases:
\citep{chen2010human}
The Human Oral Microbiome Database: a web accessible resource for investigating oral microbe taxonomic and genomic information

\citep{griffen2011core}
CORE: a phylogenetically-curated 16S rDNA database of the core oral microbiome

\citep{srinivasan2012bacterial}


\section{Phylogenetic challenges and opportunities in human microbiome research}

Usual challenges.
Non-overlapping reads.
Assemblies.

However, the apparent success of 16s tree based comparisons leads to some interesting questions.

16s tree has taken us remarkably far, but it's clearly limited.
Continued research on the correlation between phylogeny and function.
Can we correct the 16s tree somehow, or what sort of thing can we use that is a better predictor?

There is a strange divide between trait evolution methods for macro- and micro-fauna.
The results of simple methods are reasonable.
Possible that improved methods, perhaps involving whole-genome evolutionary modeling, could shed light on the problem.

There continues to be discussion about whether there is a tree of life.
Examples.
This can be a theoretical one, but here is a practical formalism: what is the correct representation of the genetic ancestry of an organism that allows us to best predict.

In any case, what about trait evolution models that take HGT into account?
What about trait evolution models that are on KEGG?

Hubbell's neutral theory applied to human microbiome \citep{fierer2012animalcules,costello2012application}.

Functional genes and phylogenomics.

The field has already benefited significantly from phylogeny using 16S.
Now we have more full-genome data.
We also have more full-genome methods.
What can we do with that?

HGT.
What about tree models of within-organism transfer?
Could we do tree shape statistics?


\section{Discussion}
What we can expect next.

Future will give deeper sequencing, which can mean massive parallelization.
Citizen microbiome projects.

Comparative studies of microbiomes.
\citep{phillips2012microbiome}
Microbiome analysis among bats describes influences of host phylogeny, life history, physiology and geography

Clinical applications,
\citep{clarridge2004}
{{I}mpact of 16{S} r{RNA} gene sequence analysis for identification of bacteria on clinical microbiology and infectious diseases}
For routine identification of a single microbe in culture, MALDI-TOF kicks ass and is cheap.
Diagnostics involving mixed communities.
Bioinformatics will get hardened into fixed workflows.

Understand cause and effect of diseases.
Take an ecololgical view on atopic dermititis.

Perhaps there is a role for trait evolution methods, or perhaps a simple method does prediction to within how good the data is.
There is the possibility that we don't have to think about individual organisms at all, but rather can think about the collection of genes and their metabolic network as a meta-organism (cite Elbo).
Has limitation that you forget about cellular boundaries, which matter-- not a freely diffusing soup.
Question: to what extent is modeling cellular-level evolutionary history interesting?
As an extension microbes have to be close to one another-- requires population modeling.

There are limitations to what we can learn.
Genome doesn't tell all.
Host side is sometimes missing, but experiments are good.
As described above, we can expect much more development in terms of specifics.
\citep{hooper2012interactions}

The field has experienced a frenetic rate of expansion over the past xxx years, and sometimes it can feel like researchers in this area have forgotten some of the basic principles of study design, such as power calculations, that are well recognized in other areas of medical research.
Although the human microbiome has led to a lot of "hype," our microbes are here to stay and so is research on them.
Thus we can look forward to the field of human microbiome analysis settling down to a comfortable and mature middle age as an interesting intersection between ecology and medicine.


\section{Acknowledgements}
Aaron Darling, Connor McCoy,
FHCRC folk.
ATD grant.


\notforarxiv{
\newpage
\section{Figure Legends}
%\FIGmassTransport
\clearpage

\newpage
}

\bibliographystyle{plainnat}
\bibliography{sbreview}

\end{document}


