
\documentclass{amsart}
\usepackage{amsmath,amsfonts,amssymb,amsthm}
\usepackage[english]{babel}
\usepackage{graphicx}
\usepackage{url}
\usepackage{palatino}
\usepackage[round]{natbib}
\newcommand{\forarxiv}[1]{#1}
\newcommand{\notforarxiv}[1]{}

% show keys for eqs, etc.
% \usepackage[notref,notcite]{showkeys}

% code
\newcommand{\program}{\textsf{program}}

\newcommand{\eat}[1]{}

\newcommand{\FIGmassTransport}{\
\begin{figure}[ht]
\begin{center}
  \forarxiv{\includegraphics[width=13cm]{mass_transport.pdf}}
\end{center}
\caption{\
  Caption goes here.
}
\label{FIGmassTransport}
\end{figure}
}
\newcommand{\refFIGmassTransport}{1}

% Noah suggests making a table for
%alpha diversity
%beta diversity
%PCA
%taxonomic assignment/classification
%clustering/binning

\begin{document}

\notforarxiv{
\begin{flushright}
Version dated: \today
\end{flushright}
\bigskip
\noindent RH: PHYLOGENETICS AND THE HUMAN MICROBIOME
\bigskip
\medskip
\begin{center}

\noindent{\Large \bf Phylogenetics and the human microbiome.}
\bigskip

\noindent {\normalsize \sc
Frederick A. Matsen IV$^1$}\\
\noindent {\small \it
$^1$
Program in Computational Biology, Fred Hutchinson Cancer Research Center, Seattle, WA, 91802, USA}\\
\end{center}
\medskip
\noindent{\bf Corresponding author:} Frederick A Matsen, Program in Computational Biology, Fred Hutchinson Cancer Research Center, Seattle, WA, 91802, USA; E-mail: matsen@fhcrc.org.\\
\vspace{1in}
}

\forarxiv{\
\title{Phylogenetics and the human microbiome.}
\author{Frederick A. Matsen IV}
\date{\today}
\begin{abstract}
}
\notforarxiv{
\subsubsection{Abstract}
}

The human microbiome is the collection of microbes that live inside and on the outside of humans.
Recent work including the Human Microbiome Project (HMP) and the Metagenomics of the Human Intestinal Tract (MetaHIT) consortia has advanced human microboime research by generating thousands of 16s surveys and terabases of metagenomic data on the human microbiome.
In this paper I review the impact of phylogenetics and tree-thinking on the methods used in the analysis of human microbiome data, as well as describing current challenges for phylogenetics coming from this type of work.

\forarxiv{
\end{abstract}
\maketitle
}

\notforarxiv{
\noindent (Keywords: human microbiome; microbial ecology; phylogenetic methods; review)\\
\vspace{1.5in}
}


\section{Introduction}

Why, as systematic biologists, should we be interested in what these microbial ecologists have done?
System with a lot of data and a lot of questions.
Microbes can only be identified by their molecular sequence.
Indeed, just to do ecology you have to do sequencing.
Many of the issues that are raised by high throughput sequencing of beasties that are more commonly of interest for phylogenetics folk are present, and worse, in microbial studies.

Taxonomic issues.
Lack of "obvious" ways of grouping beasties.
Lack of broad phenotypic characterization.
In the absence of sequencing, microbiologist have to rely on microscopes and staining.
Lots of little things look alike.
These issues motivate sequence- and evolution-based methodology.

This has led to innovation in ways of using trees.
It means something that they have developed their own way of using trees, although there is some overlap with previous literature.
Questions of microbial genomics, translated into questions of function, have important consequences for human health.

Why human microbiome versus microbial ecology?
Relevance to health.
We know a lot more about these bugs.
Can manipulate the systems-- feeding studies and gnotobiotic mice.

In this review I will describe human microbiome research from the perspective of a phylogenetics researcher.
I will first briefly review the recent literature on the human microbiome, then describe novel ways in which human microbiome researchers have used trees and the challenges that human microbiome research poses for the phylogenetics community.
I will finish with opportunities for SB audience to contribute to this field.

\section{The human microbiome}
The human microbiome is the collection of microbial organisms that live inside of and on the surface of humans.
In this section I will review what is known about them and their effects.
More bacteria than human cells.

The microbiome has remarkable metabolic potential.
\cite{qin2010human}
% To understand the impact of gut microbes on human health and well-being it is crucial to assess their genetic potential. Here we describe the Illumina-based metagenomic sequencing, assembly and characterization of 3.3 million non-redundant microbial genes, derived from 576.7 gigabases of sequence, from faecal samples of 124 European individuals. The gene set, ~150 times larger than the human gene complement, contains an overwhelming majority of the prevalent (more frequent) microbial genes of the cohort and probably includes a large proportion of the prevalent human intestinal microbial genes. The genes are largely shared among individuals of the cohort. Over 99% of the genes are bacterial, indicating that the entire cohort harbours between 1,000 and 1,150 prevalent bacterial species and each individual at least 160 such species, which are also largely shared. We define and describe the minimal gut metagenome and the minimal gut bacterial genome in terms of functions present in all individuals and most bacteria, respectively.
The microbiome is dynamic in terms of representation but apparently constant in terms of function.
\cite{hmp2012structure}
Core microbiome \cite{turnbaugh2008core}.

Microbiome is an ecosystem.
It is remarkably diverse, though less diverse than soil.
Human microbiome is spatially organized, as can be seen on skin \cite{grice2009topographical}.
Bacterial community variation in human body habitats across space and time \cite{costello2009bacterial}.
Inter-individual versus intra-individual variation \cite{hmp2012structure}.

When we perturb the microbiome, sometimes it makes fundamental changes.
Antibiotics fundamentally disturb microbial communities, resulting in an effect that lasts years.
\cite{jernberg2007long,dethlefsen2008pervasive,jakobsson2010short,dethlefsen2011incomplete}

Sometimes these changes lead to phenotypic changes in the host.
There is strong evidence of interaction between the gut microbiome and obesity, but the story is yet to be fully explored.
An intervention study has established human gut microbes associated with obesity \cite{ley2006microbial}.
An obese phenotype can be transferred from mouse to mouse by gut microbiome transplantation \cite{turnbaugh2006obesity}.
Pregnant human gut microbiome \cite{koren2012host} leads to metabolic disorder in mice.

These findings have yet to find their way into a real understanding.
Amish study \cite{zupancic2012analysis}
"Neither BMI nor any metabolic syndrome trait was associated with a particular gut community. Network analysis identified twenty-two bacterial species and four OTUs that were either positively or inversely correlated with metabolic syndrome traits, suggesting that certain members of the gut microbiota may play a role in these metabolic derangements."

more virome
http://www.pnas.org/content/early/2013/07/03/1300833110.short
review paper:http://genomebiology.com/2013/14/7/127

mycobiome
http://genomemedicine.com/content/5/7/63


In order to really get an understanding, researchers are shifting from community dynamics to fine-scale interactions between host and microbe \cite{hooper2012interactions}.
Some major findings there.

So far we have been focusing on bacteria, though archaea also play a role.
Archaea have also been found in the gut.
and fungi too
http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0066019

Although the focus of much of the work has been on bacteria and archaea, other groups of life have attracted more attention lately, such as the fungi.

Viruses too
febrile study \cite{wylie2012sequence}


\section{Methods used to investigate the human microbiome via sequencing.}

Terminology: survey versus metagenome.
Many of these methods cross over with microbial ecology in general \citep{hugenholtz2002review} (cite this?).
I will not be talking about manipulations of these systems, such as gnotobiotic mice and human intervention.


\subsection{Microbial community estimation using marker genes}

Our modern knowledge of the microbial world is due to a large part derived from the methods of Carl Woese and colleagues who pioneered the use of marker genes as a way to distinguish between microbial lineages.
Their work, and the scientists who followed them, used especially the 16S ribosomal gene.
This gene was chosen because...
They developed the first "universal" 16S PCR primers.

Non-culturable organisms.

While Woese and colleagues used primitive sequencing methods to gain information, modern researchers have the luxury of high throughput sequencing.
This can be done with a high level of multiplexing, making an explicit trade-off between depth of sequencing for each specimen and the number of specimens able to be put on the sequencer at the same time.
This leads to the question of how many sequences are needed to characterize the microbial diversity of a given environment.
For simply separating two rather different samples, rather few samples are needed \cite{kuczynski2010microbial}.

Despite the high-throughput and low cost of modern sequencing, inherent challenges remain for the use of population census by marker gene sequencing.
Most fundamentally, various microbes have different DNA extraction efficiencies, even with hard core protocols, meaning that the representation of marker gene sequences is not representative of the actual communities \cite{morgan2010metagenomic}.
Furthermore, primer bias.
Current sequencing technology is limited to a length that is shorter than most genes, which limits the resolution of the analyses.
Different regions may be good for different groups of organisms.

16S exists in multiple copies and there can be diversity within the copies.
\citet{kembel2012incorporating}

Some groups have reported advantages to using alternate single-copy genes as markers, (e.g. \cite{case2007rpob},\cite{mcnabb2004hsp65}), however nothing has really come close to 16S.


\subsection{Metagenomes}

As described above ``metagenome" means that data is sheared randomly.
This type of data has a complication above that taken from 16S survey data, which is that the genetic region sequenced is unknown in addition to the organism it was taken from.

Some use metagenomic data as an expanded marker gene library.
\cite{kembel2011phylogenetic}
AMPHORA, Phylosift.
One approach to using metagenomic data is simply to subset to genes that are found in a large proportion of micro-organisms.
This is the approach taken by \citep{von2007quantitative,stark2010mltreemap} and Jonathan Eisen \citep{wu2008amphora} (in preparation).
Because of the diversity of gene repertoire in microbes, the gene sets may have only limited overlap with one another, and even the largest collection of genes in these databases only recruits around 1 percent of a metagenome.

An alternative approach is to use metagenomic reads directly for analysis.
The authors of the metAMOS pipeline \cite{treangen2013metamos} report speedups and much higher accuracy when reads are assembled before they are classified.

If not used as an expanded marker gene panel then typically used to infer information about metabolic capacity (cite Humann) rather than make statements of an evolutionary nature.


\subsection{Data generated from the various studies}
reference genomes
HMP DACC.

HMP nature paper
\cite{methe2012framework}



\section{Tree-thinking in human microbiome research}

Phylogenetics is primarily used in this realm as a framework with which data can be organized.
It is common for researchers to equate 16S phylogeny with taxonomy.
UniFrac is interesting.

\subsection{Phylogenetics and taxonomy}

Phylogenetics has had a substantial impact on microbial ecology research by changing our view of the taxonomic relationships between microorganisms.
The most stunning example of this is the discovery that archaea, although morphologically similar to bacteria, form their own separate lineage.

However, the reason people are attached to taxonomy is that there is a lot of information using taxonomic labels.

In general, this sort of approach has led to a number projects that use phylogeny to revise taxonomy.
These attempts are less ambitious than the development of the PhyloCode \cite{forey2001phylocode}, and simply work to revise the hierarchical structure of the taxonomy while (for the most part) leaving taxonomic names fixed.
Bergey's Manual of Systematic Bacteriology has adopted 16S \cite{kreig1984bergey}.
The GreenGenes taxonomy \cite{desantis2006greengenes} has been very active in updating their taxonomy according to 16S, first with their GRUNT tool \cite{dalevi2007automated} and more recently with their tax2tree tool \cite{mcdonald2011improved}.
Rather than proposing changes to the taxonomy, our group \cite{matsen2011reconciling} has developed ways of quantifying discordance between phylogeny and taxonomy.

Tree-based classification.
\cite{bazinet2012comparative}

A fascinating approach is that of \citet{segata2012metagenomic}, who start by compiling a database of clade-specific genes \citep{segata2011metagenomic}.
Then they classify the origin of a given read as being the only clade that has this gene.
They show that this has good specificity, however, sensitivity is limited to genomes that have been sequenced.


\subsection{Diversity estimates using phylogenetics}
Because 16s surveys are inherently complex and noisy data, summary statistics are often used; summaries of the diversity of a single sample are often called \emph{alpha diversity}.
For the most part, this literature adapts methods from the classical ecological literature.

Here we will focus on variants of phylogenetic diversity and their use.

\cite{hill1973diversity}
Diversity and evenness: a unifying notation and its consequences

On expanded marker tree \cite{kembel2011phylogenetic}.

hill numbers
\cite{chao2010phylogenetic}

\cite{odwyer2012phylogenetic}
Phylogenetic Diversity Theory Sheds Light on the Structure of Microbial Communities

Despite this work, and the enthusiasm with which microbial ecologists have accepted between-community comparison, phylogenetic alpha diversity seems under-developed.
We have recently shown that partially-weighted abundance diversity measures do a good job of distinguishing between dysbiotic and "normal" states \cite{mccoy2013abundance}.
We have also looked at PD under rarefaction \cite{nipperess2013mean}.


\subsection{Community comparison using phylogenetics}

The level of similarity between samples or groups thereof are often called \emph{beta diversity};
Traditional way: \cite{jaccard1908nouvelles} Nouvelles recherches sur la distribution florale

UniFrac distance and variants has been very successful.
\cite{LozuponeKnightUniFrac05}
\cite{LozuponeEaWeightedUnifrac07}
\cite{kuczynski2010microbial} Microbial community resemblance methods differ in their ability to detect biologically relevant patterns

It recovers gradients.
\cite{nemergut2011global}
?? Global patterns in the biogeography of bacterial taxa

Figure: example principal coordinates plot.

This has precedents.
\cite{evans2012phylogenetic}
The phylogenetic Kantorovich-Rubinstein metric for environmental sequence samples

\cite{BikEaMicrobiotaStomach06}
\cite{PurdomAnalyzingDataGraphs08}

\cite{chen2012associating}
Associating microbiome composition with environment-- generalized unifrac

\cite{matsen2013edge}
Edge principal components and squash clustering: using the special structure of phylogenetic placement data for sample comparison



\subsection{Phylogeny and function}

As we will see below, 16S is frequently used as a way of measuring diversity.
Those accustomed to microbial genetics may think this surprising, because the genetic repertoire of microbes may not be so much vertically inherited.
However, \citep{zaneveld2010ribosomal} have shown that there is a correlation between pairwise distances of 16S genes and genetic repertoire.
This approach has recently been taken to its logical conclusion by trying to infer functional characteristics using discrete trait evolution models on 16S gene trees \citep{langille2013predictive} using either parsimony \citep{kluge1969quantitative} or likelihood \citep{pagel1994detecting} methods via the ape package \citep{paradis2004ape}.


\subsection{Genome-scale inquiries using phylogenetics}

A phylogeny-driven genomic encyclopaedia of Bacteria and Archaea \cite{wu2009phylogeny}.

Most wanted.

Sushi paper, network of gene transfer.

Reconciliation, team phyldog.


\section{Phylogenetic inference as practiced by human microbiome researchers}

\subsection{Alignment and tree inference}

Very large data sets.

Some groups are not interested in accuracy per se.

Alignment.
\cite{eddy1998profile}
Profile hidden Markov models.

\cite{nawrocki2009infernal}
Infernal 1.0: inference of RNA alignments

PYNAST


\cite{felsenstein1981evolutionary}
{Evolutionary trees from DNA sequences: a maximum likelihood approach}

NJ
\cite{price2010fasttree}
{FastTree 2--approximately maximum-likelihood trees for large alignments}

Species delineation problem not explicitly considered. Clustering.
\cite{li2006cdhit}
Li, W.  and Godzik, A.
\cite{edgar2010usearch}
\cite{navlakha2009finding} Finding biologically accurate clusterings in hierarchical tree decompositions using the variation of information
\cite{white2010alignment} Alignment and clustering of phylogenetic markers-implications for microbial diversity studies

Networks not considered.

Chimeric sequences are a real problem, and similar to recombination.

ARB software.
Phylogenetic placement.

\cite{wu2008simple}
A simple, fast, and accurate method of phylogenomic inference

\cite{matsen2010pplacer}

\cite{ludwig2004arb}
{{ARB}: a software environment for sequence data}
\cite{berger2011performance}

\cite{berger2011aligning}
Aligning short reads to reference alignments and trees

\cite{mirarabsepp}
{SEPP: SAT{\'e}-Enabled Phylogenetic Placement}

\cite{monierEaLargeViruses08}
{Taxonomic distribution of large DNA viruses in the sea}

\cite{stark2010mltreemap}
{{MLTreeMap}-accurate Maximum Likelihood placement of environmental DNA sequences into taxonomic and functional reference phylogenies.}

\cite{vonMeringEaQuantitative08}
{Quantitative phylogenetic assessment of microbial communities in diverse environments}


\cite{huson2007megan}

\cite{wang2007naive}
{Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy}

\cite{werner2011impact}
Impact of training sets on classification of high-throughput bacterial 16s rRNA gene surveys



\subsection{Software}
\cite{caporaso2010qiime}

\cite{schloss2009introducing}
Introducing mothur: open-source, platform-independen

\cite{lynch2013axiome}
AXIOME: automated exploration of microbial diversity

% databases
\cite{chen2010human}
The Human Oral Microbiome Database: a web accessible resource for investigating oral microbe taxonomic and genomic information

\cite{griffen2011core}
CORE: a phylogenetically-curated 16S rDNA database of the core oral microbiome

\cite{srinivasan2012bacterial}


\section{Phylogenetic challenges and opportunities in human microbiome research}

Usual challenges.
Non-overlapping reads.
Assemblies.

However, the apparent success of 16s tree based comparisons leads to some interesting questions.

16s tree has taken us remarkably far, but it's clearly limited.
Continued research on the correlation between phylogeny and function.
Can we correct the 16s tree somehow, or what sort of thing can we use that is a better predictor?

There is a strange divide between trait evolution methods for macro- and micro-fauna.
The results of simple methods are reasonable.
Possible that improved methods, perhaps involving whole-genome evolutionary modeling, could shed light on the problem.

There continues to be discussion about whether there is a tree of life.
Examples.
This can be a theoretical one, but here is a practical formalism: what is the correct representation of the genetic ancestry of an organism that allows us to best predict.

In any case, what about trait evolution models that take HGT into account?
What about trait evolution models that are on KEGG?

Hubbell's neutral theory applied to human microbiome \cite{fierer2012animalcules,costello2012application}.

Functional genes and phylogenomics.

The field has already benefited significantly from phylogeny using 16S.
Now we have more full-genome data.
We also have more full-genome methods.
What can we do with that?

HGT.
What about tree models of within-organism transfer?
Could we do tree shape statistics?


\section{Discussion}
What we can expect next.

But in addition to deeper probing we can also expect wider coverage.
Lots of data.
Citizen microbiome projects.

Comparative studies of microbiomes.
\cite{phillips2012microbiome}
Microbiome analysis among bats describes influences of host phylogeny, life history, physiology and geography

Clinical applications,
\cite{clarridge2004}
{{I}mpact of 16{S} r{RNA} gene sequence analysis for identification of bacteria on clinical microbiology and infectious diseases}
For routine identification of a single microbe in culture, MALDI-TOF kicks ass and is cheap.
Diagnostics involving mixed communities.
Bioinformatics will get hardened into fixed workflows.

Understand cause and effect of diseases.
Take an ecololgical view on atopic dermititis.

Microbial outbreak tracking.
Very common in RNA viruses.
Already using similar methods by applying whole-genome sequencing.

Perhaps there is a role for trait evolution methods, or perhaps a simple method does prediction to within how good the data is.
There is the possibility that we don't have to think about individual organisms at all, but rather can think about the collection of genes and their metabolic network as a meta-organism (cite Elbo).
Has limitation that you forget about cellular boundaries, which matter-- not a freely diffusing soup.
Question: to what extent is modeling cellular-level evolutionary history interesting?
As an extension microbes have to be close to one another-- requires population modeling.

There are limitations to what we can learn.
Genome doesn't tell all.
Host side is sometimes missing, but experiments are good.
As described above, we can expect much more development in terms of specifics.
\cite{hooper2012interactions}

The field has experienced a frenetic rate of expansion over the past xxx years, and sometimes it can feel like researchers in this area have forgotten some of the basic principles of study design, such as power calculations, that are well recognized in other areas of medical research.
Although the human microbiome has led to a lot of "hype," our microbes are here to stay and so is research on them.
Thus we can look forward to the field of human microbiome analysis settling down to a comfortable and mature middle age as an interesting intersection between ecology and medicine.


\section{Acknowledgements}
Aaron Darling, Connor McCoy,
FHCRC folk.
ATD grant.


\notforarxiv{
\newpage
\section{Figure Legends}
%\FIGmassTransport
\clearpage

\newpage
}

\bibliographystyle{plainnat}
\bibliography{sbreview}

\end{document}


