%For arXiv, uncomment this block and comment the next block.
\documentclass{amsart}
\usepackage{amsmath,amsfonts,amssymb,amsthm}
\usepackage[english]{babel}
\usepackage{graphicx}
\usepackage{url}
\usepackage[round]{natbib}
\newcommand{\forarxiv}[1]{#1}
\newcommand{\notforarxiv}[1]{}

% % For Sys Bio, uncomment this block and comment the previous block.
% \documentclass[12pt,letterpaper]{article}
% \usepackage{fixltx2e}
% \usepackage{textcomp}
% \usepackage{fullpage}
% \usepackage{amsfonts}
% \usepackage{verbatim}
% \usepackage[english]{babel}
% \usepackage{pifont}
% \usepackage{color}
% \usepackage{setspace}
% \usepackage{lscape}
% \usepackage{indentfirst}
% \usepackage[normalem]{ulem}
% \usepackage{booktabs}
% %\usepackage{nag}
% \usepackage{natbib}
% %\usepackage{bibtex}
% \usepackage{float}
% \usepackage{latexsym}
% %\usepackage{hyperref}
% \usepackage{url}
% %\usepackage{html}
% \usepackage{hyperref}
% \usepackage{epsfig}
% \usepackage{graphicx}
% \usepackage{amssymb}
% \usepackage{amsmath}
% \usepackage{bm}
% \usepackage{array}
% %\usepackage{mhchem}
% \usepackage{ifthen}
% \usepackage{caption}
% \usepackage{hyperref}
% %\usepackage{xcolor}
% \usepackage{amsthm}
% \usepackage{amstext}
% \linespread{1.66}
% \raggedright
% \setlength{\parindent}{0.5in}
% \setcounter{secnumdepth}{0}
% \pagestyle{empty}
% \renewcommand{\section}[1]{%
% \bigskip
% \begin{center}
% \begin{Large}
% \normalfont\scshape #1
% \medskip
% \end{Large}
% \end{center}}
% \renewcommand{\subsection}[1]{%
% \bigskip
% \begin{center}
% \begin{large}
% \normalfont\itshape #1
% \end{large}
% \end{center}}
% \renewcommand{\subsubsection}[1]{%
% \vspace{2ex}
% \noindent
% \textit{#1.}---}
% \renewcommand{\tableofcontents}{}
% \bibpunct{(}{)}{;}{a}{}{,}  % this is a citation format command for natbib
% \newcommand{\forarxiv}[1]{}
% \newcommand{\notforarxiv}[1]{#1}

% show keys for eqs, etc.
% \usepackage[notref,notcite]{showkeys}

% code
\newcommand{\program}{\textsf{program}}

% theorems, etc
\newtheorem{lem}{Lemma}
\newtheorem{cor}{Corollary}
\newtheorem{prop}{Proposition}
\newtheorem{thm}{Theorem}
\newtheorem{prob}{Problem}
\newtheorem{defn}{Definition}
\newtheorem{obs}{Observation}
\newtheorem{alg}{Algorithm}

\newcommand{\eat}[1]{}



\newcommand{\FIGmassTransport}{\
\begin{figure}[ht]
\begin{center}
  \forarxiv{\includegraphics[width=13cm]{mass_transport.pdf}}
\end{center}
\caption{\
  Caption goes here.
}
\label{FIGmassTransport}
\end{figure}
}
\newcommand{\refFIGmassTransport}{1}

% Noah suggests making a table for
%alpha diversity
%beta diversity
%PCA
%taxonomic assignment/classification
%clustering/binning


\begin{document}

\notforarxiv{
\begin{flushright}
Version dated: \today
\end{flushright}
\bigskip
\noindent RH: PHYLOGENETICS AND THE HUMAN MICROBIOME
\bigskip
\medskip
\begin{center}

\noindent{\Large \bf Phylogenetics and the human microbiome.}
\bigskip

\noindent {\normalsize \sc
Frederick A. Matsen IV$^1$}\\
\noindent {\small \it
$^1$
Program in Computational Biology, Fred Hutchinson Cancer Research Center, Seattle, WA, 91802, USA}\\
\end{center}
\medskip
\noindent{\bf Corresponding author:} Frederick A Matsen, Program in Computational Biology, Fred Hutchinson Cancer Research Center, Seattle, WA, 91802, USA; E-mail: matsen@fhcrc.org.\\
\vspace{1in}
}

\forarxiv{\
\title{Phylogenetics and the human microbiome.}
\author{Frederick A. Matsen IV}
\date{\today}
\begin{abstract}
}
\notforarxiv{
\subsubsection{Abstract}
}

The human microbiome is the collection of cells that live inside and on humans.
The Human Microbiome Project (HMP) and the Metagenomics of the Human Intestinal Tract (MetaHIT) consortia have advanced human microboime research by generating thousands of 16s surveys and terabases of metagenomic data on the human microbiome, as well as funding bioinformatics and statistical development.
In this paper I review the impact of phylogenetics and tree-thinking on the methods used in the analysis of this data, as well as describing current challenges for phylogenetics coming from this type of work.

\forarxiv{
\end{abstract}
\maketitle

\section{Introduction}
}

\notforarxiv{
\noindent (Keywords: human microbiome; microbial ecology; phylogenetic methods; review)\\
\vspace{1.5in}
}

\section{Introduction to the human microbiome}

Starter paragraph about the scope of human microbiome research.

We will begin by introducing the results of recent research.
% There have been hundreds of papers concerning the human microbiome, and it is not our desire to survey all of these.
% Rather, our idea is to highlight areas that may be of particular interest for the phylogenetics community by potentially having wider applications or by being areas for future exploration.
% Please note that many of the methods and issues described in this paper will be equally useful for microbial ecology in general, but by narrowing our scope we were able to keep our paper within reasonable size limits.

The human microbiome has remarkable metabolic potential.
cite qin paper, and follow up

The microbiome is dynamic in terms of representation but apparently constant in terms of function.
cites

Human microbiome is spatially organized.
cite segre,

Inter-individual versus intra-individual variation.

16s surveys have shown that antibiotics fundamentally disturb microbial communities, resulting in an effect that lasts years.
\cite{jernberg2007long,dethlefsen2008pervasive,jakobsson2010short,dethlefsen2011incomplete}

Substantial attention was given to whether microbial ``types'' exist.
A high-profile paper from Per Bork's group
http://www.nature.com/news/gut-microbial-enterotypes-become-less-clear-cut-1.10276

Important bacteria that were previously unknown?

obesity
turnbaugh papers
http://www.nature.com/nature/journal/v444/n7122/abs/nature05414.html
http://europepmc.org/abstract/MED/18407065/reload=0;jsessionid=Ea4YLfXjd7Cfpx5Hvafq.2
amish study
http://precedings.nature.com/documents/4957/version/1

Data products
reference genomes
HMP DACC.

Terminology: survey versus metagenome.
definition of a single sample.

\section{Tree-thinking in human microbiome research}

Phylogenetics is primarily used in this realm as a framework with which data can be organized.
It is common for researchers to equate 16s phylogeny with taxonomy.
Trait evolution methods are not common.

\subsection{Phylogenetics and taxonomy}

Without a doubt, the greatest impact that phylogenetics has had on microbial ecology research is on changing our view of the taxonomic relationships between microorganisms.
In the absence of sequencing, microbiologist have to rely on microscopes and staining.
Lots of little things look alike.

The most stunning example of this is the discovery of the archaea by Woese.

In general, this sort of approach has led to a number projects that use phylogeny to revise taxonomy.
These attempts are less ambitious than the development of the PhyloCode, and simply work to revise the hierarchical structure of the taxonomy while (for the most part) leaving taxonomic names fixed.
Bergey's taxonomy is supposed to be.
GreenGenes.
tax2tree.
SILVA.
%The reason people are attached to taxonomy is that there is a lot of information using taxonomic labels.

Much of this work has been done using the 16s ribosomal rRNA gene.
Exceptions.

Tree-based classification.

Curation of HM databases using phylogenetics?


\subsection{Diversity estimates using phylogenetics}

Because 16s surveys are inherently complex and noisy data, summary statistics are often used; summaries of the diversity of a single sample are often called \emph{alpha diversity}.
For the most part, this literature adapts methods from the classical ecological literature.

Here we will focus on variants of phylogenetic diversity and their use.

GEBA. Most wanted.

Eisen darling paper on extractability.


\subsection{Community comparison using phylogenetics}

The level of similarity between samples or groups thereof are often called \emph{beta diversity}; it is the current standard to employ phylogenetic tree structure in this sort of comparison.

UniFrac distance and variants.

Figure: example principal coordinates plot.


\subsection{Inference of horizontal gene transfer}




\section{Phylogenetic inference as practiced by human microbiome researchers}

Very large data sets.

Some groups are not interested in accuracy per se.

Alignment.

Networks not considered.

Species delineation problem not explicitly considered. Clustering.

Chimeric sequences are a real problem, and similar to recombination.

ARB software.
Phylogenetic placement.

Trait evolution not considered.


\section{Phylogenetic challenges for human microbiome research}

Non-overlapping reads.
Assemblies.

Functional genes and phylogenomics.

HGT.

There has been a shift from interest in community dynamics to fine-scale interactions between host and immune system.

Trait evolution methods are conspicuously absent.

\cite{wylie2012sequence}
\cite{chen2012associating}


\section{Discussion}

What we can expect next.

Field is still not yet mature.

\notforarxiv{
\newpage
\section{Figure Legends}
%\FIGmassTransport
\clearpage

\newpage
}

\bibliographystyle{plainnat}
\bibliography{sbreview}

\end{document}
