%For arXiv, uncomment this block and comment the next block.
\documentclass{amsart}
\usepackage{amsmath,amsfonts,amssymb,amsthm}
\usepackage[english]{babel}
\usepackage{graphicx}
\usepackage{url}
\usepackage[round]{natbib}
\newcommand{\forarxiv}[1]{#1}
\newcommand{\notforarxiv}[1]{}

% % For Sys Bio, uncomment this block and comment the previous block.
% \documentclass[12pt,letterpaper]{article}
% \usepackage{fixltx2e}
% \usepackage{textcomp}
% \usepackage{fullpage}
% \usepackage{amsfonts}
% \usepackage{verbatim}
% \usepackage[english]{babel}
% \usepackage{pifont}
% \usepackage{color}
% \usepackage{setspace}
% \usepackage{lscape}
% \usepackage{indentfirst}
% \usepackage[normalem]{ulem}
% \usepackage{booktabs}
% %\usepackage{nag}
% \usepackage{natbib}
% %\usepackage{bibtex}
% \usepackage{float}
% \usepackage{latexsym}
% %\usepackage{hyperref}
% \usepackage{url}
% %\usepackage{html}
% \usepackage{hyperref}
% \usepackage{epsfig}
% \usepackage{graphicx}
% \usepackage{amssymb}
% \usepackage{amsmath}
% \usepackage{bm}
% \usepackage{array}
% %\usepackage{mhchem}
% \usepackage{ifthen}
% \usepackage{caption}
% \usepackage{hyperref}
% %\usepackage{xcolor}
% \usepackage{amsthm}
% \usepackage{amstext}
% \linespread{1.66}
% \raggedright
% \setlength{\parindent}{0.5in}
% \setcounter{secnumdepth}{0}
% \pagestyle{empty}
% \renewcommand{\section}[1]{%
% \bigskip
% \begin{center}
% \begin{Large}
% \normalfont\scshape #1
% \medskip
% \end{Large}
% \end{center}}
% \renewcommand{\subsection}[1]{%
% \bigskip
% \begin{center}
% \begin{large}
% \normalfont\itshape #1
% \end{large}
% \end{center}}
% \renewcommand{\subsubsection}[1]{%
% \vspace{2ex}
% \noindent
% \textit{#1.}---}
% \renewcommand{\tableofcontents}{}
% \bibpunct{(}{)}{;}{a}{}{,}  % this is a citation format command for natbib
% \newcommand{\forarxiv}[1]{}
% \newcommand{\notforarxiv}[1]{#1}

% show keys for eqs, etc.
% \usepackage[notref,notcite]{showkeys}

% code
\newcommand{\program}{\textsf{program}}

% theorems, etc
\newtheorem{lem}{Lemma}
\newtheorem{cor}{Corollary}
\newtheorem{prop}{Proposition}
\newtheorem{thm}{Theorem}
\newtheorem{prob}{Problem}
\newtheorem{defn}{Definition}
\newtheorem{obs}{Observation}
\newtheorem{alg}{Algorithm}

\newcommand{\eat}[1]{}



\newcommand{\FIGmassTransport}{\
\begin{figure}[ht]
\begin{center}
  \forarxiv{\includegraphics[width=13cm]{mass_transport.pdf}}
\end{center}
\caption{\
  Caption goes here.
}
\label{FIGmassTransport}
\end{figure}
}
\newcommand{\refFIGmassTransport}{1}

% Noah suggests making a table for
%alpha diversity
%beta diversity
%PCA
%taxonomic assignment/classification
%clustering/binning


\begin{document}

\notforarxiv{
\begin{flushright}
Version dated: \today
\end{flushright}
\bigskip
\noindent RH: PHYLOGENETICS AND THE HUMAN MICROBIOME
\bigskip
\medskip
\begin{center}

\noindent{\Large \bf Phylogenetics and the human microbiome.}
\bigskip

\noindent {\normalsize \sc
Frederick A. Matsen IV$^1$}\\
\noindent {\small \it
$^1$
Program in Computational Biology, Fred Hutchinson Cancer Research Center, Seattle, WA, 91802, USA}\\
\end{center}
\medskip
\noindent{\bf Corresponding author:} Frederick A Matsen, Program in Computational Biology, Fred Hutchinson Cancer Research Center, Seattle, WA, 91802, USA; E-mail: matsen@fhcrc.org.\\
\vspace{1in}
}

\forarxiv{\
\title{Phylogenetics and the human microbiome.}
\author{Frederick A. Matsen IV}
\date{\today}
\begin{abstract}
}
\notforarxiv{
\subsubsection{Abstract}
}

The human microbiome is the collection of cells that live inside and on humans.
The Human Microbiome Project (HMP) and the Metagenomics of the Human Intestinal Tract (MetaHIT) consortia have advanced human microboime research by generating thousands of 16s surveys and terabases of metagenomic data on the human microbiome, as well as funding bioinformatics and statistical development.
In this paper I review the impact of phylogenetics and tree-thinking on the methods used in the analysis of this data, as well as describing current challenges for phylogenetics coming from this type of work.

\forarxiv{
\end{abstract}
\maketitle

\section{Introduction}
}

\notforarxiv{
\noindent (Keywords: human microbiome; microbial ecology; phylogenetic methods; review)\\
\vspace{1.5in}
}

\section{Introduction to the human microbiome}

Starter paragraph about the scope of human microbiome research.

We will begin by introducing the results of recent research.
% There have been hundreds of papers concerning the human microbiome, and it is not our desire to survey all of these.
% Rather, our idea is to highlight areas that may be of particular interest for the phylogenetics community by potentially having wider applications or by being areas for future exploration.
% Please note that many of the methods and issues described in this paper will be equally useful for microbial ecology in general, but by narrowing our scope we were able to keep our paper within reasonable size limits.

The human microbiome has remarkable metabolic potential.
cite qin paper, and follow up

The microbiome is dynamic in terms of representation but apparently constant in terms of function.
cites

Human microbiome is spatially organized.
cite segre,

Inter-individual versus intra-individual variation.

16s surveys have shown that antibiotics fundamentally disturb microbial communities, resulting in an effect that lasts years.
\cite{jernberg2007long,dethlefsen2008pervasive,jakobsson2010short,dethlefsen2011incomplete}

Substantial attention was given to whether microbial ``types'' exist.
A high-profile paper from Per Bork's group
http://www.nature.com/news/gut-microbial-enterotypes-become-less-clear-cut-1.10276

Important bacteria that were previously unknown?

\cite{hugenholtz2002review}
{{E}xploring prokaryotic diversity in the genomic era}


Major finding: antibiotics.
\cite{dethlefsen2008pervasive,dethlefsen2011incomplete,jakobsson2010short,jernberg2007long}
obesity
turnbaugh papers
http://www.nature.com/nature/journal/v444/n7122/abs/nature05414.html
http://europepmc.org/abstract/MED/18407065/reload=0;jsessionid=Ea4YLfXjd7Cfpx5Hvafq.2
amish study
http://precedings.nature.com/documents/4957/version/1

Data products
reference genomes
HMP DACC.

Core microbiome
\cite{turnbaugh2008core}


HMP nature paper
\cite{methe2012framework}


% applications
\cite{costello2009bacterial}
Bacterial community variation in human body habitats across space and time

\cite{ley2006microbial}
Microbial ecology: human gut microbes associated with obesity
Terminology: survey versus metagenome.
definition of a single sample.

\section{Tree-thinking in human microbiome research}

Phylogenetics is primarily used in this realm as a framework with which data can be organized.
It is common for researchers to equate 16s phylogeny with taxonomy.
Trait evolution methods are not common.

\subsection{Phylogenetics and taxonomy}

Without a doubt, the greatest impact that phylogenetics has had on microbial ecology research is on changing our view of the taxonomic relationships between microorganisms.
In the absence of sequencing, microbiologist have to rely on microscopes and staining.
Lots of little things look alike.

The most stunning example of this is the discovery of the archaea by Woese.

In general, this sort of approach has led to a number projects that use phylogeny to revise taxonomy.
These attempts are less ambitious than the development of the PhyloCode, and simply work to revise the hierarchical structure of the taxonomy while (for the most part) leaving taxonomic names fixed.
Bergey's taxonomy is supposed to be.
GreenGenes.
tax2tree.
SILVA.
%The reason people are attached to taxonomy is that there is a lot of information using taxonomic labels.

Much of this work has been done using the 16s ribosomal rRNA gene.
Exceptions.

Tree-based classification.

\cite{parks2011classifying}
Classifying short genomic fragments from novel lineages using composition and homology

\cite{mcdonald2011improved}
An improved Greengenes taxonomy with explicit ranks for ecological and evolutionary analyses of bacteria and archaea

\cite{phylopythia}
McHardy, A. C.  and Martin, H. G.  and Tsirigos, A.  and Hugenholtz, P.  and Rigoutsos, I.

\cite{munch2008statistical}
Statistical assignment of DNA sequences using Bayesian phylogenetics

\cite{munch2008fast}
Fast phylogenetic DNA barcoding

\cite{rosen2011nbc}
NBC: the Naive Bayes Classification tool webserver for taxonomic classification of metagenomic reads


\cite{segata2011metagenomic}
Metagenomic biomarker discovery and explanation

\cite{segata2012metagenomic}
Metagenomic microbial community profiling using unique clade-specific marker genes



Curation of HM databases using phylogenetics?

\cite{forey2001phylocode}
{The PhyloCode: description and commentary}

\cite{matsen2011reconciling}
Reconciling taxonomy and phylogenetic inference: formalism and algorithms for describing discord and inferring taxonomic roots

\cite{carma}


\cite{kreig1984bergey}
Bergey's Manual of Systematic Bacteriology.

\cite{dalevi2007automated}
Automated group assignment in large phylogenetic trees using GRUNT: GRouping, Ungrouping, Naming Tool

\cite{desantis2006greengenes}
{Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with ARB}

\cite{bazinet2012comparative}
A comparative evaluation of sequence classification programs

\cite{brady2011phymmbl}
PhymmBL expanded: confidence scores, custom databases, parallelization and more

\cite{brady2009phymm}
Phymm and PhymmBL: metagenomic phylogenetic classification with interpolated Markov models

\cite{ColeEaRDP2009}

\cite{liu2008accurate}
Accurate taxonomy assignments from 16S rRNA sequences produced by highly parallel pyrosequencers

\cite{lan2012using}
Using the RDP Classifier to Predict Taxonomic Novelty and Reduce the Search Space for Finding Novel Organisms

\cite{macdonald2012rapid}
Rapid identification of high-confidence taxonomic assignments for metagenomic data

\cite{maidak1997rdp}
The RDP (ribosomal database project)


\subsection{Diversity estimates using phylogenetics}

Because 16s surveys are inherently complex and noisy data, summary statistics are often used; summaries of the diversity of a single sample are often called \emph{alpha diversity}.
For the most part, this literature adapts methods from the classical ecological literature.

Here we will focus on variants of phylogenetic diversity and their use.

GEBA. Most wanted.

Eisen darling paper on extractability.

\cite{hill1973diversity}
Diversity and evenness: a unifying notation and its consequences


\cite{odwyer2012phylogenetic}
Phylogenetic Diversity Theory Sheds Light on the Structure of Microbial Communities

cite kembel paper

\cite{morlon2011spatial}
Spatial patterns of phylogenetic diversity


\subsection{Community comparison using phylogenetics}

The level of similarity between samples or groups thereof are often called \emph{beta diversity}; it is the current standard to employ phylogenetic tree structure in this sort of comparison.

\cite{chen2012associating}
Associating microbiome composition with environment-- generalized unifrac

UniFrac distance and variants.
\cite{LozuponeKnightUniFrac05}
\cite{LozuponeEaWeightedUnifrac07}

Figure: example principal coordinates plot.

\cite{BikEaMicrobiotaStomach06}

hill numbers
\cite{chao2010phylogenetic}

\cite{evans2012phylogenetic}
The phylogenetic Kantorovich-Rubinstein metric for environmental sequence samples

\cite{kuczynski2010microbial}
Microbial community resemblance methods differ in their ability to detect biologically relevant patterns


\cite{jaccard1908nouvelles}
Nouvelles recherches sur la distribution florale


\cite{matsen2013edge}
Edge principal components and squash clustering: using the special structure of phylogenetic placement data for sample comparison

\cite{PurdomAnalyzingDataGraphs08}


\cite{nemergut2011global}
?? Global patterns in the biogeography of bacterial taxa

\subsection{Inference of horizontal gene transfer}




\section{Phylogenetic inference as practiced by human microbiome researchers}

\cite{felsenstein1981evolutionary}
{Evolutionary trees from DNA sequences: a maximum likelihood approach}

\cite{eddy1998profile}
Profile hidden Markov models.

\cite{li2006cdhit}
Li, W.  and Godzik, A.
\cite{edgar2010usearch}

\cite{mirarabsepp}
{SEPP: SAT{\'e}-Enabled Phylogenetic Placement}

\cite{monierEaLargeViruses08}
{Taxonomic distribution of large DNA viruses in the sea}


\cite{huson2007megan}

\cite{mcnabb2004hsp65}
{{A}ssessment of partial sequencing of the 65-kilodalton heat shock protein gene (hsp65) for routine identification of {M}ycobacterium species isolated from clinical sources}

\cite{price2010fasttree}
{FastTree 2--approximately maximum-likelihood trees for large alignments}

\cite{navlakha2009finding}
Finding biologically accurate clusterings in hierarchical tree decompositions using the variation of information

\cite{nawrocki2009infernal}
Infernal 1.0: inference of RNA alignments

\cite{white2010alignment}
Alignment and clustering of phylogenetic markers-implications for microbial diversity studies


RNA copy number
\cite{morganEaInVitroSimulatedMetagenome10}


Very large data sets.

Some groups are not interested in accuracy per se.

Alignment.

Networks not considered.

Species delineation problem not explicitly considered. Clustering.

Chimeric sequences are a real problem, and similar to recombination.

ARB software.
Phylogenetic placement.

\cite{wu2008simple}
A simple, fast, and accurate method of phylogenomic inference

\cite{matsen2010pplacer}

\cite{ludwig2004arb}
{{ARB}: a software environment for sequence data}
\cite{berger2011performance}

\cite{berger2011aligning}
Aligning short reads to reference alignments and trees

\cite{caporaso2010qiime}

\cite{schloss2009introducing}
Introducing mothur: open-source, platform-independen

\cite{lynch2013axiome}
AXIOME: automated exploration of microbial diversity


\cite{wang2007naive}
{Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy}

\cite{werner2011impact}
Impact of training sets on classification of high-throughput bacterial 16s rRNA gene surveys


\cite{stark2010mltreemap}
{{MLTreeMap}-accurate Maximum Likelihood placement of environmental DNA sequences into taxonomic and functional reference phylogenies.}

\cite{vonMeringEaQuantitative08}
{Quantitative phylogenetic assessment of microbial communities in diverse environments}

% alternative loci
\cite{case2007rpob}

% databases
\cite{chen2010human}
The Human Oral Microbiome Database: a web accessible resource for investigating oral microbe taxonomic and genomic information

\cite{griffen2011core}
CORE: a phylogenetically-curated 16S rDNA database of the core oral microbiome

\cite{srinivasan2012bacterial}

Trait evolution not considered.


\section{Phylogenetic challenges for human microbiome research}

Non-overlapping reads.
Assemblies.

Functional genes and phylogenomics.

HGT.

There has been a shift from interest in community dynamics to fine-scale interactions between host and immune system.

Trait evolution methods are conspicuously absent.

\cite{wylie2012sequence}
\cite{chen2012associating}

% medical
\cite{clarridge2004}
{{I}mpact of 16{S} r{RNA} gene sequence analysis for identification of bacteria on clinical microbiology and infectious diseases}

\cite{phillips2012microbiome}
Microbiome analysis among bats describes influences of host phylogeny, life history, physiology and geography



\section{Discussion}

What we can expect next.

Field is still not yet mature.

\notforarxiv{
\newpage
\section{Figure Legends}
%\FIGmassTransport
\clearpage

\newpage
}

\bibliographystyle{plainnat}
\bibliography{sbreview}

\end{document}

\cite{hummelen2010deep}
Deep sequencing of the vaginal microbiota of women with HIV

\cite{ravel2011vaginal}
Vaginal microbiome of reproductive-age women

\cite{fredricks2005molecular}
Molecular identification of bacteria associated with bacterial vaginosis

